# HG changeset patch
# Parent 185ef5bee9f087458e702352afadfc7e7d87ec50

diff -r 185ef5bee9f0 archivers/pax/files/getid.c
--- a/archivers/pax/files/getid.c	Tue Oct 09 17:51:24 2012 -0400
+++ b/archivers/pax/files/getid.c	Tue Oct 09 17:51:29 2012 -0400
@@ -152,12 +152,14 @@
 	if (!grstart() || !pwstart())
 		return (0);
 
+#if 0
 				/* switch pwcache(3) lookup functions */
 	if (pwcache_groupdb(gi_setgroupent, gi_endgrent,
 			    gi_getgrnam, gi_getgrgid) == -1
 	    || pwcache_userdb(gi_setpassent, gi_endpwent,
 			    gi_getpwnam, gi_getpwuid) == -1)
 		return (0);
+#endif
 
 	return (1);
 }
diff -r 185ef5bee9f0 archivers/pax/files/options.c
--- a/archivers/pax/files/options.c	Tue Oct 09 17:51:24 2012 -0400
+++ b/archivers/pax/files/options.c	Tue Oct 09 17:51:29 2012 -0400
@@ -107,6 +107,8 @@
 #include "mtree.h"
 #endif	/* SMALL */
 
+#include <strings.h>
+
 /*
  * Routines which handle command line options
  */
