# HG changeset patch
# Parent 757f0b95fa63555ef69c9d34f1e0989a608fb402

diff -r 757f0b95fa63 bootstrap/bootstrap
--- a/bootstrap/bootstrap	Tue Oct 09 17:58:53 2012 -0400
+++ b/bootstrap/bootstrap	Tue Oct 09 17:58:58 2012 -0400
@@ -749,7 +749,7 @@
 # MACHINE_ARCH on platforms where we override bmake's value.
 OPSYS=${opsys}
 export OPSYS
-if [ "${machine_arch}" != "" ]; then
+if [ "${machine_arch}" != "" -a "${MACHINE_ARCH}" = "" ]; then
 	MACHINE_ARCH=${machine_arch}
 	export MACHINE_ARCH
 fi
