# HG changeset patch
# Parent 0bec78b4e2c0cf1b3b8c083c131218785e3614e7

diff -r 0bec78b4e2c0 chat/irssi/Makefile
--- a/chat/irssi/Makefile	Mon Apr 08 12:52:16 2013 -0400
+++ b/chat/irssi/Makefile	Mon Apr 08 12:52:20 2013 -0400
@@ -18,11 +18,16 @@
 USE_LIBTOOL=		yes
 USE_TOOLS+=		pkg-config
 
+.include "../../mk/termcap.buildlink3.mk"
+
 CONFIGURE_ARGS+=	--with-terminfo \
 			--with-proxy \
-			--without-ncurses \
 			--sysconfdir=${PKG_SYSCONFDIR}
 
+.if "${TERMCAP_TYPE}" != "curses" || "${CURSES_TYPE}" != "ncurses"
+CONFIGURE_ARGS+=	--without-ncurses
+.endif
+
 EGDIR=		${PREFIX}/share/examples/irssi
 CONF_FILES=	${EGDIR}/irssi.conf ${PKG_SYSCONFDIR}/irssi.conf
 
