# HG changeset patch
# Parent 3239c1f017c86f0191014efcd157828f0b0eeb9a

diff -r 3239c1f017c8 databases/db4/distinfo
--- a/databases/db4/distinfo	Tue Oct 09 17:51:44 2012 -0400
+++ b/databases/db4/distinfo	Tue Oct 09 17:51:49 2012 -0400
@@ -7,4 +7,5 @@
 SHA1 (patch-ab) = 8f9ecc47527d338e28ee1ce71e694eaf2bdceb92
 SHA1 (patch-ac) = 045a3b83b9c6c164382ec546b06ae248620552cf
 SHA1 (patch-ae) = 43e574154de93dba60392b09d6ebddbca1036849
+SHA1 (patch-aq) = 3baa90e922dc9cb075deee2a2a37d607affead9b
 SHA1 (patch-dbinc_atomic.h) = ff60e3af41ac1b000ebccc91bc673adde5c7463b
diff -r 3239c1f017c8 databases/db4/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/databases/db4/patches/patch-aq	Tue Oct 09 17:51:49 2012 -0400
@@ -0,0 +1,11 @@
+diff -r 7a6cd6105754 dist/configure
+--- dist/configure	Sat Apr 28 15:23:22 2012 +0000
++++ dist/configure	Sat Apr 28 15:24:05 2012 +0000
+@@ -18792,6 +18792,7 @@
+ /* end confdefs.h.  */
+ 
+ #include <pthread.h>
++#include <stdlib.h>
+ main() {
+ 	pthread_cond_t cond;
+ 	pthread_mutex_t mutex;
