# HG changeset patch
# Parent 18498942af6cb35d438c2dde54d6e7eb1d820660

diff -r 18498942af6c devel/gmake/Makefile
--- a/devel/gmake/Makefile	Mon Apr 08 12:52:37 2013 -0400
+++ b/devel/gmake/Makefile	Mon Apr 08 12:52:41 2013 -0400
@@ -42,6 +42,7 @@
 	${CHMOD} g-s ${DESTDIR}${PREFIX}/bin/gmake
 	${CHGRP} ${BINGRP} ${DESTDIR}${PREFIX}/bin/gmake
 	${LN} -sf ${PREFIX}/bin/gmake ${DESTDIR}${PREFIX}/${PKGGNUDIR}/bin/make
-	${LN} -sf ${PREFIX}/${PKGMANDIR}/man1/gmake.1 ${DESTDIR}${PREFIX}/${PKGGNUDIR}${PKGMANDIR}/man1/make.1
+	mkdir -p ${DESTDIR}${PREFIX}/${PKGGNUDIR}man/man1
+	${LN} -sf ${PREFIX}/${PKGMANDIR}/man1/gmake.1 ${DESTDIR}${PREFIX}/${PKGGNUDIR}man/man1/make.1
 
 .include "../../mk/bsd.pkg.mk"
