diff -r b89cfb80649c devel/libuuid/distinfo
--- a/devel/libuuid/distinfo	Sat Mar 14 00:03:23 2015 -0400
+++ b/devel/libuuid/distinfo	Sat Mar 14 00:03:30 2015 -0400
@@ -4,5 +4,6 @@
 RMD160 (util-linux-2.24.2.tar.xz) = 7db147ba5545aadef19920815b6461001afb1f4e
 Size (util-linux-2.24.2.tar.xz) = 3586580 bytes
 SHA1 (patch-aa) = 50054bed3cb5da7ffebb7bfd3f2da0d92b0616d7
+SHA1 (patch-aq) = f8ededcca54e2adb13f57e564623178f4d46ced6
 SHA1 (patch-ab) = 160c13ca5ff36c4ed78aaa9bd3b4d270a5a8b847
 SHA1 (patch-lib_randutils.c) = 5cab5d27a4936478c55c8d11dc002df9a0a41962
diff -r b89cfb80649c devel/libuuid/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devel/libuuid/patches/patch-aq	Sat Mar 14 00:03:30 2015 -0400
@@ -0,0 +1,56 @@
+diff -r f3d747a80d82 shlibs/uuid/src/Makefile.am
+--- shlibs/uuid/src/Makefile.am	Sun May 06 03:03:55 2012 +0000
++++ shlibs/uuid/src/Makefile.am	Sun May 06 03:04:28 2012 +0000
+@@ -25,15 +25,15 @@
+ 
+ # move lib from $(usrlib_execdir) to $(libdir) if needed
+ install-exec-hook:
+-	if test "$(usrlib_execdir)" != "$(libdir)"; then \
+-		mkdir -p $(DESTDIR)$(libdir); \
+-		mv $(DESTDIR)$(usrlib_execdir)/libuuid.so.* $(DESTDIR)$(libdir); \
+-		so_img_name=$$(readlink $(DESTDIR)$(usrlib_execdir)/libuuid.so); \
+-		so_img_rel_target=$$(echo $(usrlib_execdir) | sed 's,\(^/\|\)[^/][^/]*,..,g'); \
+-		(cd $(DESTDIR)$(usrlib_execdir) && \
+-			rm -f libuuid.so && \
+-			$(LN_S) $$so_img_rel_target$(libdir)/$$so_img_name libuuid.so); \
+-	fi
++#	if test "$(usrlib_execdir)" != "$(libdir)"; then \
++#		mkdir -p $(DESTDIR)$(libdir); \
++#		mv $(DESTDIR)$(usrlib_execdir)/libuuid.so.* $(DESTDIR)$(libdir); \
++#		so_img_name=$$(readlink $(DESTDIR)$(usrlib_execdir)/libuuid.so); \
++#		so_img_rel_target=$$(echo $(usrlib_execdir) | sed 's,\(^/\|\)[^/][^/]*,..,g'); \
++#		(cd $(DESTDIR)$(usrlib_execdir) && \
++#			rm -f libuuid.so && \
++#			$(LN_S) $$so_img_rel_target$(libdir)/$$so_img_name libuuid.so); \
++#	fi
+ 
+ uninstall-hook:
+ 	rm -f $(DESTDIR)$(libdir)/libuuid.so*
+diff -r f3d747a80d82 shlibs/uuid/src/Makefile.in
+--- shlibs/uuid/src/Makefile.in	Sun May 06 03:03:55 2012 +0000
++++ shlibs/uuid/src/Makefile.in	Sun May 06 03:04:28 2012 +0000
+@@ -700,15 +700,15 @@
+ 
+ # move lib from $(usrlib_execdir) to $(libdir) if needed
+ install-exec-hook:
+-	if test "$(usrlib_execdir)" != "$(libdir)"; then \
+-		mkdir -p $(DESTDIR)$(libdir); \
+-		mv $(DESTDIR)$(usrlib_execdir)/libuuid.so.* $(DESTDIR)$(libdir); \
+-		so_img_name=$$(readlink $(DESTDIR)$(usrlib_execdir)/libuuid.so); \
+-		so_img_rel_target=$$(echo $(usrlib_execdir) | sed 's,\(^/\|\)[^/][^/]*,..,g'); \
+-		(cd $(DESTDIR)$(usrlib_execdir) && \
+-			rm -f libuuid.so && \
+-			$(LN_S) $$so_img_rel_target$(libdir)/$$so_img_name libuuid.so); \
+-	fi
++#	if test "$(usrlib_execdir)" != "$(libdir)"; then \
++#		mkdir -p $(DESTDIR)$(libdir); \
++#		mv $(DESTDIR)$(usrlib_execdir)/libuuid.so.* $(DESTDIR)$(libdir); \
++#		so_img_name=$$(readlink $(DESTDIR)$(usrlib_execdir)/libuuid.so); \
++#		so_img_rel_target=$$(echo $(usrlib_execdir) | sed 's,\(^/\|\)[^/][^/]*,..,g'); \
++#		(cd $(DESTDIR)$(usrlib_execdir) && \
++#			rm -f libuuid.so && \
++#			$(LN_S) $$so_img_rel_target$(libdir)/$$so_img_name libuuid.so); \
++#	fi
+ 
+ uninstall-hook:
+ 	rm -f $(DESTDIR)$(libdir)/libuuid.so*
