# HG changeset patch
# Parent 7556b0dd00180684436eb843e2521a6a8f3e0424

diff -r 7556b0dd0018 devel/imake/Makefile
--- a/devel/imake/Makefile	Mon Feb 18 12:23:48 2013 -0500
+++ b/devel/imake/Makefile	Mon Feb 18 12:23:52 2013 -0500
@@ -41,6 +41,9 @@
 
 .include "../../meta-pkgs/modular-xorg/hacks.mk"
 
+pre-build:
+	sed 's/-x c -/-x c - -x none/' -i ${WRKSRC}/imake.c
+
 .include "../../x11/xproto/buildlink3.mk"
 
 .include "../../mk/bsd.pkg.mk"
