diff -r d422434ec002 devel/cmake/distinfo
--- a/devel/cmake/distinfo	Sat Mar 14 00:08:55 2015 -0400
+++ b/devel/cmake/distinfo	Sat Mar 14 00:09:02 2015 -0400
@@ -13,3 +13,4 @@
 SHA1 (patch-aa) = 97bfad3d2c357e9af01677ba86057b78f0661b9b
 SHA1 (patch-ab) = aee3fb2f908aed1ce6e92b7c7ccf5b06f0596502
 SHA1 (patch-ac) = fe679b28793d6a7b9fd41cbcb7288295a4c4a508
+SHA1 (patch-qq) = bedbdcd003ee66b793fbbebcfe338b34783f938f
diff -r d422434ec002 devel/cmake/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devel/cmake/patches/patch-qq	Sat Mar 14 00:09:02 2015 -0400
@@ -0,0 +1,12 @@
+diff -r 2741184fed90 Source/kwsys/SystemInformation.cxx
+--- Source/kwsys/SystemInformation.cxx	Fri Sep 20 20:39:00 2013 +0000
++++ Source/kwsys/SystemInformation.cxx	Fri Sep 20 20:42:08 2013 +0000
+@@ -130,7 +130,7 @@
+ #   define KWSYS_SYSTEMINFORMATION_IMPLEMENT_FQDN
+ #  endif
+ # endif
+-# if defined(__GNUC__)
++# if defined(__GLIBC__)
+ #  include <execinfo.h>
+ #  if !(defined(__LSB_VERSION__) && __LSB_VERSION__ < 41)
+ #   define KWSYS_SYSTEMINFORMATION_HAVE_BACKTRACE
