# HG changeset patch
# Parent 0151d8f0d1f4b8b95310dff76dffd603f67086ee

diff -r 0151d8f0d1f4 devel/nss/Makefile
--- a/devel/nss/Makefile	Tue Oct 09 17:52:52 2012 -0400
+++ b/devel/nss/Makefile	Tue Oct 09 17:52:57 2012 -0400
@@ -55,6 +55,17 @@
 	find ${WRKSRC} -type f | xargs ${CHMOD} 644
 	find ${WRKSRC} -type d | xargs ${CHMOD} 755
 
+# [Gregor] And just where is this type SUPPOSED to come from?
+_WRAP_EXTRA_ARGS.CC+=-Duint=uint32_t
+
+# [Gregor] This build loses, but needs, our CFLAGS
+_WRAP_EXTRA_ARGS.CC+=-D_GNU_SOURCE -D_BSD_SOURCE
+
+# [Gregor] More misdetection
+pre-build:
+	${SED} 's/DHAVE_SYS_CDEFS_H/UHAVE_SYS_CDEFS_H/' -i ${WRKSRC}/security/dbm/config/config.mk
+	${SED} 's/(ffs/(__builtin_ffs/g' -i ${WRKSRC}/security/nss/cmd/checkcert/checkcert.c
+
 do-build:
 	cd ${WRKSRC}/security/nss && ${SETENV} ${MAKE_ENV} ${MAKE_PROGRAM} \
 	  build_coreconf build_dbm all
