# HG changeset patch
# Parent f13c9a0df242885807fe605270afbe3327f22340

diff -r f13c9a0df242 devel/bison/Makefile
--- a/devel/bison/Makefile	Tue Oct 09 17:51:50 2012 -0400
+++ b/devel/bison/Makefile	Tue Oct 09 17:51:55 2012 -0400
@@ -26,6 +26,7 @@
 # Avoid rebuilding manpage
 pre-build:
 	${TOUCH} ${WRKSRC}/doc/bison.1
+	sed -i 's/#elif 0/#elif 1/' -i ${WRKSRC}/lib/fseterr.c
 
 # "bison" wants a recent version of "gettext" which at least some
 # NetBSD versions don't provide. Figure out whether it will install
