# HG changeset patch
# Parent 0e69533ff3b7f37d98a124e69fc743e77aa1369c

diff -r 0e69533ff3b7 devel/gmake/Makefile
--- a/devel/gmake/Makefile	Tue Oct 09 17:52:09 2012 -0400
+++ b/devel/gmake/Makefile	Tue Oct 09 17:52:14 2012 -0400
@@ -39,6 +39,8 @@
 CONFIGURE_ARGS+=	ac_cv_func_strndup=no
 .endif
 
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -D__BEOS__"
+
 post-install:
 	${CHMOD} g-s ${DESTDIR}${PREFIX}/bin/gmake
 	${CHGRP} ${BINGRP} ${DESTDIR}${PREFIX}/bin/gmake
