# HG changeset patch
# Parent f818293ceb5c77380c5291395bf0138e99ed0e9c
diff --git a/devel/xulrunner/Makefile b/devel/xulrunner/Makefile
--- a/devel/xulrunner/Makefile
+++ b/devel/xulrunner/Makefile
@@ -34,6 +34,9 @@
 .include "mozilla-common.mk"
 .include "options.mk"
 
+# [Gregor] To make some patches work
+CFLAGS+=		-D__musl__
+
 pre-configure:
 	cd ${WRKSRC} && autoconf
 	cd ${WRKSRC}/js/src && autoconf
