# HG changeset patch
# Parent e86f9346276891fd03415e607441a06171ccc508

diff -r e86f93462768 devel/boehm-gc/Makefile
--- a/devel/boehm-gc/Makefile	Sat Mar 14 00:07:09 2015 -0400
+++ b/devel/boehm-gc/Makefile	Sat Mar 14 00:07:16 2015 -0400
@@ -27,6 +27,9 @@
 CFLAGS+=		-D_XOPEN_SOURCE
 .endif
 
+# [Gregor] Without this, needs libc-specific data in (glibc) link.h
+CFLAGS+=	-DUSE_PROC_FOR_LIBRARIES -DNO_GETCONTEXT
+
 .include "options.mk"
 
 post-install:
diff -r e86f93462768 devel/boehm-gc/distinfo
--- a/devel/boehm-gc/distinfo	Sat Mar 14 00:07:09 2015 -0400
+++ b/devel/boehm-gc/distinfo	Sat Mar 14 00:07:16 2015 -0400
@@ -4,3 +4,4 @@
 RMD160 (gc-7.4.2.tar.gz) = 106bda88d4e47c32f5ba4137c82fa173313d09a9
 Size (gc-7.4.2.tar.gz) = 1082597 bytes
 SHA1 (patch-dyn__load.c) = cd224eb9e31deba2f5a29f3202d8ef130e8fdaae
+SHA1 (patch-aq) = 89bbe084f50c7f0e457c7a221b4bafab38a77982
diff -r e86f93462768 devel/boehm-gc/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devel/boehm-gc/patches/patch-aq	Sat Mar 14 00:07:16 2015 -0400
@@ -0,0 +1,22 @@
+diff -r 748b34e952a7 dyn_load.c
+--- dyn_load.c	Mon Aug 20 22:47:08 2012 +0000
++++ dyn_load.c	Mon Aug 20 22:49:09 2012 +0000
+@@ -97,6 +97,7 @@
+     /* If you don't need the "dynamic loading" feature, you may build   */
+     /* the collector with -D IGNORE_DYNAMIC_LOADING.                    */
+ #   include <linker.h>
++# elif USE_PROC_FOR_LIBRARIES
+ # else
+ #   include <link.h>
+ # endif
+diff -r 748b34e952a7 os_dep.c
+--- os_dep.c	Mon Aug 20 22:47:08 2012 +0000
++++ os_dep.c	Mon Aug 20 22:49:09 2012 +0000
+@@ -26,6 +26,7 @@
+ #   define __KERNEL__
+ #   include <asm/signal.h>
+ #   undef __KERNEL__
++# elif 1
+ # else
+     /* Kernels prior to 2.1.1 defined struct sigcontext_struct instead of */
+     /* struct sigcontext.  libc6 (glibc2) uses "struct sigcontext" in     */
