# HG changeset patch
# Parent 174875b66779ac14b580cc8b843344abc35becf1

diff -r 174875b66779 devel/flex/PLIST
--- a/devel/flex/PLIST	Mon Jul 08 19:19:53 2013 -0400
+++ b/devel/flex/PLIST	Mon Jul 08 19:19:58 2013 -0400
@@ -13,20 +13,3 @@
 share/doc/flex/README
 share/doc/flex/README.cvs
 share/doc/flex/TODO
-share/locale/ca/LC_MESSAGES/flex.mo
-share/locale/da/LC_MESSAGES/flex.mo
-share/locale/de/LC_MESSAGES/flex.mo
-share/locale/es/LC_MESSAGES/flex.mo
-share/locale/fi/LC_MESSAGES/flex.mo
-share/locale/fr/LC_MESSAGES/flex.mo
-share/locale/ga/LC_MESSAGES/flex.mo
-share/locale/ko/LC_MESSAGES/flex.mo
-share/locale/nl/LC_MESSAGES/flex.mo
-share/locale/pl/LC_MESSAGES/flex.mo
-share/locale/pt_BR/LC_MESSAGES/flex.mo
-share/locale/ro/LC_MESSAGES/flex.mo
-share/locale/ru/LC_MESSAGES/flex.mo
-share/locale/sv/LC_MESSAGES/flex.mo
-share/locale/tr/LC_MESSAGES/flex.mo
-share/locale/vi/LC_MESSAGES/flex.mo
-share/locale/zh_CN/LC_MESSAGES/flex.mo
