# HG changeset patch
# Parent 95928b2a6cc264560d02d14d4260d00818bafe15

diff -r 95928b2a6cc2 devel/pcre/Makefile
--- a/devel/pcre/Makefile	Tue Oct 09 17:52:58 2012 -0400
+++ b/devel/pcre/Makefile	Tue Oct 09 17:53:03 2012 -0400
@@ -20,6 +20,8 @@
 CONFIGURE_ARGS+=	--enable-utf8 --enable-unicode-properties
 CONFIGURE_ARGS+=	--enable-pcre16
 
+CONFIGURE_ARGS+=	--disable-cpp
+
 SUBST_CLASSES+=		rpath
 SUBST_STAGE.rpath=	pre-configure
 SUBST_MESSAGE.rpath=	Fixing rpath output in pcre-config
diff -r 95928b2a6cc2 devel/pcre/PLIST
--- a/devel/pcre/PLIST	Tue Oct 09 17:52:58 2012 -0400
+++ b/devel/pcre/PLIST	Tue Oct 09 17:53:03 2012 -0400
@@ -3,18 +3,12 @@
 bin/pcregrep
 bin/pcretest
 include/pcre.h
-include/pcre_scanner.h
-include/pcre_stringpiece.h
-include/pcrecpp.h
-include/pcrecpparg.h
 include/pcreposix.h
 lib/libpcre.la
 lib/libpcre16.la
-lib/libpcrecpp.la
 lib/libpcreposix.la
 lib/pkgconfig/libpcre.pc
 lib/pkgconfig/libpcre16.pc
-lib/pkgconfig/libpcrecpp.pc
 lib/pkgconfig/libpcreposix.pc
 man/man1/pcre-config.1
 man/man1/pcregrep.1
@@ -75,7 +69,6 @@
 man/man3/pcrebuild.3
 man/man3/pcrecallout.3
 man/man3/pcrecompat.3
-man/man3/pcrecpp.3
 man/man3/pcrejit.3
 man/man3/pcrelimits.3
 man/man3/pcrematching.3
@@ -127,7 +120,6 @@
 share/doc/pcre/html/pcrebuild.html
 share/doc/pcre/html/pcrecallout.html
 share/doc/pcre/html/pcrecompat.html
-share/doc/pcre/html/pcrecpp.html
 share/doc/pcre/html/pcredemo.html
 share/doc/pcre/html/pcregrep.html
 share/doc/pcre/html/pcrejit.html
