# HG changeset patch
# Parent 78ad6a1d3ec80f7428a3fed2e074b1057f57b6eb

diff -r 78ad6a1d3ec8 devel/xulrunner/Makefile
--- a/devel/xulrunner/Makefile	Mon Feb 18 12:29:24 2013 -0500
+++ b/devel/xulrunner/Makefile	Mon Feb 18 12:29:28 2013 -0500
@@ -39,6 +39,9 @@
 PLIST.sps=	yes
 .endif
 
+# [Gregor] To make some patches work
+CFLAGS+=		-D__musl__
+
 pre-configure:
 	cd ${WRKSRC} && autoconf
 	cd ${WRKSRC}/js/src && autoconf
