# HG changeset patch
# Parent 36a33cc9fdc71ad78d9752c2b0a18b9f9f88bfa9
diff -r 36a33cc9fdc7 -r fbf8ed8bcb35 devel/glib2/distinfo
--- a/devel/glib2/distinfo	Wed Jul 31 15:32:26 2013 -0400
+++ b/devel/glib2/distinfo	Wed Jul 31 19:18:11 2013 -0400
@@ -34,3 +34,4 @@
 SHA1 (patch-glib_gfileutils.c) = 667e9ed2a8a9cca09f9eb4218e467d11544822f3
 SHA1 (patch-glib_gmarkup.c) = 238cce5f20d2fd9a12832d47d498edfd7290075b
 SHA1 (patch-glib_gstrfuncs.c) = 6719b83e35e1f4e4e8b1c113faa8596676fd9ae5
+SHA1 (patch-qq) = bde0378f7e5966bc445dc25aa7de96529583cd8e
diff -r 36a33cc9fdc7 -r fbf8ed8bcb35 devel/glib2/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devel/glib2/patches/patch-qq	Wed Jul 31 19:18:11 2013 -0400
@@ -0,0 +1,59 @@
+diff -r 9e8d43ec61dc configure
+--- configure	Wed Jul 31 16:56:24 2013 +0000
++++ configure	Wed Jul 31 16:57:42 2013 +0000
+@@ -21745,7 +21745,7 @@
+ glib_save_CFLAGS=$CFLAGS
+ if test "x$GCC" = "xyes"; then :
+ 
+-  CFLAGS="$CFLAGS -Werror"
++  #CFLAGS="$CFLAGS -Werror"
+   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+ /* end confdefs.h.  */
+ #include <stdlib.h>
+@@ -28857,10 +28857,7 @@
+ 
+ 
+   for flag in \
+-   -Wall -Wstrict-prototypes -Werror=declaration-after-statement \
+-   -Werror=missing-prototypes -Werror=implicit-function-declaration \
+-   -Werror=pointer-arith -Werror=init-self -Werror=format-security \
+-   -Werror=format=2 -Werror=missing-include-dirs; do
++   -Wall -Wstrict-prototypes; do
+ 
+   { $as_echo "$as_me:${as_lineno-$LINENO}: checking if $CC supports flag $flag in envvar CFLAGS" >&5
+ $as_echo_n "checking if $CC supports flag $flag in envvar CFLAGS... " >&6; }
+diff -r f47e85ebc1d7 gio/gfile.c
+--- gio/gfile.c	Wed Jul 31 17:04:30 2013 +0000
++++ gio/gfile.c	Wed Jul 31 17:06:39 2013 +0000
+@@ -22,6 +22,8 @@
+  * Author: Alexander Larsson <alexl@redhat.com>
+  */
+ 
++#define _GNU_SOURCE /* for splice */
++
+ #include "config.h"
+ 
+ #ifdef __linux__
+@@ -2821,9 +2823,9 @@
+ 
+ static gboolean
+ do_splice (int     fd_in,
+-           loff_t *off_in,
++           off_t *off_in,
+            int     fd_out,
+-           loff_t *off_out,
++           off_t *off_out,
+            size_t  len,
+            long   *bytes_transferd,
+            GError **error)
+@@ -2866,8 +2868,8 @@
+   int buffer[2] = { -1, -1 };
+   gboolean res;
+   goffset total_size;
+-  loff_t offset_in;
+-  loff_t offset_out;
++  off_t offset_in;
++  off_t offset_out;
+   int fd_in, fd_out;
+ 
+   fd_in = g_file_descriptor_based_get_fd (G_FILE_DESCRIPTOR_BASED (in));
