# HG changeset patch
# Parent f49ad3be0f45bdc998d76b1e49eb800835105f51

diff -r f49ad3be0f45 devel/bison/Makefile
--- a/devel/bison/Makefile	Mon Feb 18 12:23:28 2013 -0500
+++ b/devel/bison/Makefile	Mon Feb 18 12:23:32 2013 -0500
@@ -25,6 +25,7 @@
 # Avoid rebuilding manpage
 pre-build:
 	${TOUCH} ${WRKSRC}/doc/bison.1
+	sed -i 's/#elif 0/#elif 1/' -i ${WRKSRC}/lib/fseterr.c
 
 # "bison" wants a recent version of "gettext" which at least some
 # NetBSD versions don't provide. Figure out whether it will install
