# HG changeset patch
# Parent 5a5fa1f2958b178ec26f84e8aca5cbe1afbc3352

diff -r 5a5fa1f2958b devel/gmake/Makefile
--- a/devel/gmake/Makefile	Mon Feb 18 12:23:43 2013 -0500
+++ b/devel/gmake/Makefile	Mon Feb 18 12:23:48 2013 -0500
@@ -38,6 +38,8 @@
 CONFIGURE_ARGS+=	ac_cv_func_strndup=no
 .endif
 
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -D__BEOS__"
+
 post-install:
 	${CHMOD} g-s ${DESTDIR}${PREFIX}/bin/gmake
 	${CHGRP} ${BINGRP} ${DESTDIR}${PREFIX}/bin/gmake
