# HG changeset patch
# Parent 449406e316f7bcf81709655aff593a8093cffe7a

diff -r 449406e316f7 devel/boehm-gc/Makefile
--- a/devel/boehm-gc/Makefile	Mon Feb 18 12:30:07 2013 -0500
+++ b/devel/boehm-gc/Makefile	Mon Feb 18 12:30:11 2013 -0500
@@ -39,6 +39,9 @@
 CFLAGS+=		-D_XOPEN_SOURCE
 .endif
 
+# [Gregor] Without this, needs libc-specific data in (glibc) link.h
+CFLAGS+=	-DUSE_PROC_FOR_LIBRARIES -DNO_GETCONTEXT
+
 .include "options.mk"
 
 post-install:
diff -r 449406e316f7 devel/boehm-gc/distinfo
--- a/devel/boehm-gc/distinfo	Mon Feb 18 12:30:07 2013 -0500
+++ b/devel/boehm-gc/distinfo	Mon Feb 18 12:30:11 2013 -0500
@@ -4,3 +4,4 @@
 RMD160 (gc-7.2d.tar.gz) = e3e191de01cc0f5b3b2570842e897c11fcc8d96d
 Size (gc-7.2d.tar.gz) = 1263064 bytes
 SHA1 (patch-ab) = b595e5fb653e078189ad6aadfd932b0c0e9fff4a
+SHA1 (patch-aq) = 89bbe084f50c7f0e457c7a221b4bafab38a77982
diff -r 449406e316f7 devel/boehm-gc/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devel/boehm-gc/patches/patch-aq	Mon Feb 18 12:30:11 2013 -0500
@@ -0,0 +1,22 @@
+diff -r 748b34e952a7 dyn_load.c
+--- dyn_load.c	Mon Aug 20 22:47:08 2012 +0000
++++ dyn_load.c	Mon Aug 20 22:49:09 2012 +0000
+@@ -97,6 +97,7 @@
+     /* If you don't need the "dynamic loading" feature, you may build   */
+     /* the collector with -D IGNORE_DYNAMIC_LOADING.                    */
+ #   include <linker.h>
++# elif USE_PROC_FOR_LIBRARIES
+ # else
+ #   include <link.h>
+ # endif
+diff -r 748b34e952a7 os_dep.c
+--- os_dep.c	Mon Aug 20 22:47:08 2012 +0000
++++ os_dep.c	Mon Aug 20 22:49:09 2012 +0000
+@@ -26,6 +26,7 @@
+ #   define __KERNEL__
+ #   include <asm/signal.h>
+ #   undef __KERNEL__
++# elif 1
+ # else
+     /* Kernels prior to 2.1.1 defined struct sigcontext_struct instead of */
+     /* struct sigcontext.  libc6 (glibc2) uses "struct sigcontext" in     */
