# HG changeset patch
# Parent 2bf7a01a41488b61d2ce6bbcc189fe90a05942e2
diff -r 2bf7a01a4148 -r bf0ec68bd79a devel/gmake/Makefile
--- a/devel/gmake/Makefile	Wed Mar 13 20:09:14 2013 +0000
+++ b/devel/gmake/Makefile	Wed Mar 13 20:37:32 2013 +0000
@@ -42,6 +42,7 @@
 	${CHMOD} g-s ${DESTDIR}${PREFIX}/bin/gmake
 	${CHGRP} ${BINGRP} ${DESTDIR}${PREFIX}/bin/gmake
 	${LN} -sf ${PREFIX}/bin/gmake ${DESTDIR}${PREFIX}/${PKGGNUDIR}/bin/make
-	${LN} -sf ${PREFIX}/${PKGMANDIR}/man1/gmake.1 ${DESTDIR}${PREFIX}/${PKGGNUDIR}${PKGMANDIR}/man1/make.1
+	mkdir -p ${DESTDIR}${PREFIX}/${PKGGNUDIR}man/man1
+	${LN} -sf ${PREFIX}/${PKGMANDIR}/man1/gmake.1 ${DESTDIR}${PREFIX}/${PKGGNUDIR}man/man1/make.1
 
 .include "../../mk/bsd.pkg.mk"
