# HG changeset patch
# Parent 966a907cecee42f05a6e844f8fae58ae04fcd072

diff -r 966a907cecee editors/vim-share/Makefile.common
--- a/editors/vim-share/Makefile.common	Tue Oct 09 17:53:04 2012 -0400
+++ b/editors/vim-share/Makefile.common	Tue Oct 09 17:53:09 2012 -0400
@@ -72,5 +72,8 @@
 ALTERNATIVES_SRC=
 .endif
 
+# [Gregor] vim doesn't recognize that it needs curses to build ANYTHING, even the share component
+.include "../../mk/curses.buildlink3.mk"
+
 post-patch:
 	find ${WRKDIR} -name '*.orig_dist' -exec rm -f {} \;
