# HG changeset patch
# Parent 299a6df5893d702ee5590455c2f6a45fe72b9d07

diff -r 299a6df5893d editors/vim-share/Makefile.common
--- a/editors/vim-share/Makefile.common	Sat Mar 14 00:03:30 2015 -0400
+++ b/editors/vim-share/Makefile.common	Sat Mar 14 00:03:36 2015 -0400
@@ -55,5 +55,8 @@
 ALTERNATIVES_SRC=
 .endif
 
+# [Gregor] vim doesn't recognize that it needs curses to build ANYTHING, even the share component
+.include "../../mk/curses.buildlink3.mk"
+
 post-patch:
 	find ${WRKDIR} -name '*.orig_dist' -exec rm -f {} \;
