# HG changeset patch
# Parent a8f8a87a5e233fa063b2d157efa265a816506058

diff -r a8f8a87a5e23 editors/vim-share/Makefile.common
--- a/editors/vim-share/Makefile.common	Mon Apr 08 12:52:58 2013 -0400
+++ b/editors/vim-share/Makefile.common	Mon Apr 08 12:53:02 2013 -0400
@@ -56,5 +56,8 @@
 ALTERNATIVES_SRC=
 .endif
 
+# [Gregor] vim doesn't recognize that it needs curses to build ANYTHING, even the share component
+.include "../../mk/curses.buildlink3.mk"
+
 post-patch:
 	find ${WRKDIR} -name '*.orig_dist' -exec rm -f {} \;
