# HG changeset patch
# Parent 6f2af86c637bd26c8aa2fce8a97364f232581ede

diff -r 6f2af86c637b editors/vim-share/Makefile.common
--- a/editors/vim-share/Makefile.common	Mon Feb 18 12:24:29 2013 -0500
+++ b/editors/vim-share/Makefile.common	Mon Feb 18 12:24:33 2013 -0500
@@ -56,5 +56,8 @@
 ALTERNATIVES_SRC=
 .endif
 
+# [Gregor] vim doesn't recognize that it needs curses to build ANYTHING, even the share component
+.include "../../mk/curses.buildlink3.mk"
+
 post-patch:
 	find ${WRKDIR} -name '*.orig_dist' -exec rm -f {} \;
