# HG changeset patch
# Parent 6b42b3a1d79f0ae7d94ab164eee8c3e4dcb8a2d0

diff -r 6b42b3a1d79f filesystems/fuse/distinfo
--- a/filesystems/fuse/distinfo	Sat Mar 14 00:03:36 2015 -0400
+++ b/filesystems/fuse/distinfo	Sat Mar 14 00:03:43 2015 -0400
@@ -4,3 +4,4 @@
 RMD160 (fuse-2.9.3.tar.gz) = a81f350842a43aa82cff2cb75e0bf9f5b123ad5a
 Size (fuse-2.9.3.tar.gz) = 572044 bytes
 SHA1 (patch-aa) = 1179876c7a4f230d8df8a7a6724b90779c9fa0e3
+SHA1 (patch-qq) = f8dc23f988fa537dcb41bdc54f4dc550eb479914
diff -r 6b42b3a1d79f filesystems/fuse/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/filesystems/fuse/patches/patch-qq	Sat Mar 14 00:03:43 2015 -0400
@@ -0,0 +1,48 @@
+diff -r 18a7de07ebb3 example/fioclient.c
+--- example/fioclient.c	Tue May 08 17:27:11 2012 +0000
++++ example/fioclient.c	Tue May 08 17:34:04 2012 +0000
+@@ -10,7 +10,7 @@
+ */
+ 
+ #include <sys/types.h>
+-#include <sys/fcntl.h>
++#include <fcntl.h>
+ #include <sys/stat.h>
+ #include <sys/ioctl.h>
+ #include <stdio.h>
+diff -r 18a7de07ebb3 lib/mount_util.c
+--- lib/mount_util.c	Tue May 08 17:27:11 2012 +0000
++++ lib/mount_util.c	Tue May 08 17:34:04 2012 +0000
+@@ -24,6 +24,12 @@
+ #include <sys/mount.h>
+ #include <sys/param.h>
+ 
++#ifndef _PATH_MOUNTED
++#ifdef MOUNTED
++#define _PATH_MOUNTED MOUNTED
++#endif
++#endif
++
+ #ifdef __NetBSD__
+ #define umount2(mnt, flags) unmount(mnt, (flags == 2) ? MNT_FORCE : 0)
+ #define mtab_needs_update(mnt) 0
+diff -r 18a7de07ebb3 util/fusermount.c
+--- util/fusermount.c	Tue May 08 17:27:11 2012 +0000
++++ util/fusermount.c	Tue May 08 17:34:04 2012 +0000
+@@ -29,6 +29,16 @@
+ #include <sys/utsname.h>
+ #include <sched.h>
+ 
++#ifndef _PATH_MOUNTED
++#ifdef MOUNTED
++#define _PATH_MOUNTED MOUNTED
++#endif
++#endif
++
++#ifndef __WCLONE
++#define __WCLONE      0x80000000
++#endif
++
+ #define FUSE_COMMFD_ENV		"_FUSE_COMMFD"
+ 
+ #define FUSE_DEV_OLD "/proc/fs/fuse/dev"
