# HG changeset patch
# Parent f0afd2a0bcf3c7f7ac44d35f53aae671fea605c4

diff -r f0afd2a0bcf3 graphics/gimp/PLIST
--- a/graphics/gimp/PLIST	Tue Oct 09 17:53:40 2012 -0400
+++ b/graphics/gimp/PLIST	Tue Oct 09 17:53:45 2012 -0400
@@ -316,7 +316,6 @@
 lib/gimp/2.0/plug-ins/file-tiff-load
 lib/gimp/2.0/plug-ins/file-tiff-save
 lib/gimp/2.0/plug-ins/file-uri
-lib/gimp/2.0/plug-ins/file-wmf
 lib/gimp/2.0/plug-ins/file-xbm
 lib/gimp/2.0/plug-ins/file-xjt
 ${PLIST.remote}lib/gimp/2.0/plug-ins/file-xmc
