# HG changeset patch
# Parent b1438460cda64851e1d5b4c2edf6a8b505f17dd1

diff -r b1438460cda6 graphics/gimp/PLIST
--- a/graphics/gimp/PLIST	Sat Mar 14 00:03:43 2015 -0400
+++ b/graphics/gimp/PLIST	Sat Mar 14 00:03:50 2015 -0400
@@ -317,7 +317,6 @@
 lib/gimp/2.0/plug-ins/file-tiff-load
 lib/gimp/2.0/plug-ins/file-tiff-save
 lib/gimp/2.0/plug-ins/file-uri
-lib/gimp/2.0/plug-ins/file-wmf
 lib/gimp/2.0/plug-ins/file-xbm
 lib/gimp/2.0/plug-ins/file-xjt
 ${PLIST.remote}lib/gimp/2.0/plug-ins/file-xmc
diff -r b1438460cda6 graphics/gimp/distinfo
--- a/graphics/gimp/distinfo	Sat Mar 14 00:03:43 2015 -0400
+++ b/graphics/gimp/distinfo	Sat Mar 14 00:03:50 2015 -0400
@@ -11,3 +11,4 @@
 SHA1 (patch-devel-docs_app_Makefile.in) = 89aec477783899cc6a5ac765d567d73f79474d5e
 SHA1 (patch-plug-ins_common_file-mng.c) = 8ed244997cdef7125e1aa5da4d0fac5068ad5159
 SHA1 (patch-plug-ins_script-fu_script-fu-server.c) = c424b1a8e345ec3d0763ac2cf1f8535da9e10797
+SHA1 (patch-qq) = 89fd0ce763a2668861a90a2e7c129e4c94b644cf
diff -r b1438460cda6 graphics/gimp/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/graphics/gimp/patches/patch-qq	Sat Mar 14 00:03:50 2015 -0400
@@ -0,0 +1,22 @@
+diff -r f497c2200f57 app/base/base-utils.c
+--- app/base/base-utils.c	Thu Nov 22 06:07:46 2012 +0000
++++ app/base/base-utils.c	Thu Nov 22 06:31:37 2012 +0000
+@@ -33,7 +33,8 @@
+ /* For get_backtrace() */
+ #include <stdlib.h>
+ #include <string.h>
+-#ifdef __linux__
++#if defined(__linux__) && defined(__GLIBC__)
++#define HAVE_EXECINFO_H 1
+ #include <execinfo.h>
+ #endif
+ #endif
+@@ -114,7 +115,7 @@
+ char *
+ get_backtrace (void)
+ {
+-#ifdef __linux__
++#ifdef HAVE_EXECINFO_H
+   void     *functions[MAX_FUNC];
+   char    **function_names;
+   int       n_functions;
