# HG changeset patch
# Parent 5d8f3490a566c51f8d823ee31deaa738171f850a

diff -r 5d8f3490a566 graphics/cairo/Makefile
--- a/graphics/cairo/Makefile	Tue Oct 09 17:53:28 2012 -0400
+++ b/graphics/cairo/Makefile	Tue Oct 09 17:53:33 2012 -0400
@@ -38,6 +38,8 @@
 # For snprintf() and ctime_r()
 CPPFLAGS.SunOS+=		-D__EXTENSIONS__ -D_POSIX_PTHREAD_SEMANTICS
 
+CPPFLAGS+=		-Dffs=__builtin_ffs
+
 TEST_TARGET=			check
 
 # PR#43928
