# HG changeset patch
# Parent 83bb198078aab1aceb8b590e51162acd5d7c847b

diff -r 83bb198078aa graphics/MesaLib/distinfo
--- a/graphics/MesaLib/distinfo	Mon Jul 08 19:20:30 2013 -0400
+++ b/graphics/MesaLib/distinfo	Mon Jul 08 19:20:36 2013 -0400
@@ -30,3 +30,4 @@
 SHA1 (patch-src_mesa_main_compiler.h) = b164781b541cb83b64597facbb7a7e21d8bdb686
 SHA1 (patch-src_mesa_main_imports.c) = d0c50004c749e4c5c136ac2ed50a6989ca90065f
 SHA1 (patch-src_mesa_main_imports.h) = f5a96195b3a14294ca3389c9fe17a6076c46db8a
+SHA1 (patch-savage_sys_timeb) = ac4304c7060d4c191a557c7017cc630a0de6a318
diff -r 83bb198078aa graphics/MesaLib/patches/patch-savage_sys_timeb
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/graphics/MesaLib/patches/patch-savage_sys_timeb	Mon Jul 08 19:20:36 2013 -0400
@@ -0,0 +1,13 @@
+sys/timeb.h is unportable and not even used
+
+diff -r 4a616c1c5444 src/mesa/drivers/dri/savage/savageioctl.c
+--- src/mesa/drivers/dri/savage/savageioctl.c	Wed Mar 13 22:28:00 2013 +0000
++++ src/mesa/drivers/dri/savage/savageioctl.c	Wed Mar 13 22:42:15 2013 +0000
+@@ -43,7 +43,6 @@
+ 
+ #include "drm.h"
+ #include <sys/ioctl.h>
+-#include <sys/timeb.h>
+ 
+ #define DEPTH_SCALE_16 ((1<<16)-1)
+ #define DEPTH_SCALE_24 ((1<<24)-1)
