# HG changeset patch
# Parent 11483e6494c8fbd9cbe29b65cadb062001f08a7f

diff -r 11483e6494c8 graphics/MesaLib/Makefile
--- a/graphics/MesaLib/Makefile	Tue Oct 09 17:53:16 2012 -0400
+++ b/graphics/MesaLib/Makefile	Tue Oct 09 17:53:21 2012 -0400
@@ -72,6 +72,7 @@
 
 pre-build:
 	cd ${WRKSRC} && ${RM} -fr src/glu src/glut progs
+	sed -i 's/#if defined(__linux__) && defined(__i386__)/#if 0/' ${WRKSRC}/src/mesa/main/glheader.h
 
 .include "../../mk/compiler.mk"
 
