# HG changeset patch
# Parent b9dff25c6dad1d93f649662930a6af0b9e4cce96

diff -r b9dff25c6dad graphics/cairo/Makefile
--- a/graphics/cairo/Makefile	Mon Feb 18 12:24:50 2013 -0500
+++ b/graphics/cairo/Makefile	Mon Feb 18 12:24:55 2013 -0500
@@ -35,6 +35,8 @@
 # For snprintf() and ctime_r()
 CPPFLAGS.SunOS+=		-D__EXTENSIONS__ -D_POSIX_PTHREAD_SEMANTICS
 
+CPPFLAGS+=		-Dffs=__builtin_ffs
+
 TEST_TARGET=			check
 
 # PR#43928
