# HG changeset patch
# Parent 26e0e50ab7361d5574bc76001f72ee0581431638

diff -r 26e0e50ab736 lang/python27/PLIST.common
--- a/lang/python27/PLIST.common	Sat Mar 14 00:03:49 2015 -0400
+++ b/lang/python27/PLIST.common	Sat Mar 14 00:03:56 2015 -0400
@@ -1928,7 +1928,6 @@
 lib/python${PY_VER_SUFFIX}/lib-dynload/itertools.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/math.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/mmap.so
-${PLIST.nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis.so
 ${PLIST.no-nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis_failed.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/operator.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/parser.so
