# HG changeset patch
# Parent bee42adf1b7ead3146b642ad3d9e85e53e6c0c3a

diff -r bee42adf1b7e lang/python27/PLIST.common
--- a/lang/python27/PLIST.common	Mon Apr 08 12:53:38 2013 -0400
+++ b/lang/python27/PLIST.common	Mon Apr 08 12:53:43 2013 -0400
@@ -1836,7 +1836,6 @@
 lib/python${PY_VER_SUFFIX}/lib-dynload/itertools.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/math.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/mmap.so
-${PLIST.nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis.so
 ${PLIST.no-nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis_failed.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/operator.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/parser.so
