# HG changeset patch
# Parent 0fe2b31d22d1368cfd64be73161e00aa3a102d36

diff -r 0fe2b31d22d1 lang/perl5/Makefile
--- a/lang/perl5/Makefile	Mon Feb 18 12:25:16 2013 -0500
+++ b/lang/perl5/Makefile	Mon Feb 18 12:25:20 2013 -0500
@@ -46,6 +46,7 @@
 HAS_CONFIGURE=		yes
 CONFIGURE_SCRIPT=	./Configure
 CONFIGURE_ARGS+=	-sde
+CONFIGURE_ARGS+=        -D inc_version_list=none
 MAKE_ENV+=		LC_ALL="C"
 
 TEST_TARGET=		test
@@ -203,6 +204,10 @@
 
 pre-configure:
 	cd ${WRKSRC} && find `pwd` -name "*.orig" -type f -exec ${RM} -f {} \;
+	sed 's/-fstack-protector/-fno-stack-protector/g' -i ${WRKSRC}/Configure
+	
+pre-build:	
+	sed 's/define HAS_CRYPT_R/undef HAS_CRYPT_R/g' -i ${WRKSRC}/config.h
 
 post-build:
 	${SED}	-e "s,@PERL5@,"${PERL5:Q}",g"				\
