# HG changeset patch
# Parent b2b1567e9ae0acfa4a66a39f9477c5cce89b52f7

diff -r b2b1567e9ae0 lang/python26/Makefile
--- a/lang/python26/Makefile	Mon Feb 18 12:25:21 2013 -0500
+++ b/lang/python26/Makefile	Mon Feb 18 12:25:25 2013 -0500
@@ -115,22 +115,7 @@
 .endif
 
 PLIST_VARS+=	dll nis no-nis
-.if ${OPSYS} == "IRIX"
-.  if ${ABI} == "64"
-PLIST.no-nis=	yes
-.  else
-PLIST.nis=	yes
-.  endif
-.else
 PLIST.dll=	yes
-.  if ${OPSYS} == "MirBSD"
-# neither nis nor no-nis
-.  elif ${OPSYS} != "NetBSD" || exists(/usr/bin/ypcat)
-PLIST.nis=	yes
-.  else
-PLIST.no-nis=	yes
-.  endif
-.endif
 
 .include "../../mk/bdb.buildlink3.mk"
 MAKE_ENV+=	PY_BDB_TYPE=${BDB_TYPE}
