# HG changeset patch
# Parent 11041fc6aa31eec8c7b2d5568d7f40c6142af01a

diff -r 11041fc6aa31 lang/python27/PLIST.common
--- a/lang/python27/PLIST.common	Mon Jul 08 19:21:01 2013 -0400
+++ b/lang/python27/PLIST.common	Mon Jul 08 19:21:07 2013 -0400
@@ -1848,7 +1848,6 @@
 lib/python${PY_VER_SUFFIX}/lib-dynload/itertools.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/math.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/mmap.so
-${PLIST.nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis.so
 ${PLIST.no-nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis_failed.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/operator.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/parser.so
