# HG changeset patch
# Parent e7efc0b94b96d2b7e3e1e6a4b72714884b72f175

diff -r e7efc0b94b96 lang/python27/PLIST.common
--- a/lang/python27/PLIST.common	Mon Feb 18 12:25:26 2013 -0500
+++ b/lang/python27/PLIST.common	Mon Feb 18 12:25:30 2013 -0500
@@ -1836,7 +1836,6 @@
 lib/python${PY_VER_SUFFIX}/lib-dynload/itertools.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/math.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/mmap.so
-${PLIST.nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis.so
 ${PLIST.no-nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis_failed.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/operator.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/parser.so
