# HG changeset patch
# Parent d8f49598f472d1e45825769c5a540caa2bfec97f

diff -r d8f49598f472 lang/php53/Makefile
--- a/lang/php53/Makefile	Mon Feb 18 12:30:18 2013 -0500
+++ b/lang/php53/Makefile	Mon Feb 18 12:30:22 2013 -0500
@@ -54,5 +54,8 @@
 	cd ${WRKSRC}; ${INSTALL_DATA} php.ini-development php.ini-production \
 		${DESTDIR}${EGDIR}
 	${INSTALL_DATA} ${WRKSRC}/php.gif ${DESTDIR}${PREFIX}/share/php
-
+	
+# [Gregor] misdetection
+CONFIGURE_ENV+=	ac_cv_type_uint=yes ac_cv_type_ulong=yes	
+	
 .include "../../mk/bsd.pkg.mk"
