# HG changeset patch
# Parent 06802205ddc3a5ae2a0e5dc984edca3681033b35

diff -r 06802205ddc3 lang/ruby193-base/Makefile
--- a/lang/ruby193-base/Makefile	Sat Mar 14 00:03:56 2015 -0400
+++ b/lang/ruby193-base/Makefile	Sat Mar 14 00:04:02 2015 -0400
@@ -174,6 +174,13 @@
 		${DESTDIR}${PREFIX}/${RUBY_EG}/openssl
 .endfor
 	${RUBY_GENERATE_PLIST}
+	
+# [Gregor] docs fail for unknown reasons
+CONFIGURE_ARGS+=	--disable-install-doc
+CONFIGURE_ENV+=	LDSHARED="${CC} -shared"
+
+# [Gregor] bug with dln.o
+CFLAGS+=-fPIC
 
 .include "../../mk/bdb.buildlink3.mk"
 .include "../../converters/libiconv/buildlink3.mk"
diff -r 06802205ddc3 lang/ruby193-base/distinfo
--- a/lang/ruby193-base/distinfo	Sat Mar 14 00:03:56 2015 -0400
+++ b/lang/ruby193-base/distinfo	Sat Mar 14 00:04:02 2015 -0400
@@ -44,3 +44,4 @@
 SHA1 (patch-test_rubygems_test__gem.rb) = fe8b8e436b255595122d2a01462ba0a686575f24
 SHA1 (patch-tool_rbinstall.rb) = f0497085a09b31145f57028b30e335b09acabcff
 SHA1 (patch-vsnprintf.c) = 7bb4ca07381536717518143ef7803634168ca9fa
+SHA1 (patch-qq) = 7f4871f89582ebf99f4119a5397720c83a1603fd
diff -r 06802205ddc3 lang/ruby193-base/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/lang/ruby193-base/patches/patch-qq	Sat Mar 14 00:04:02 2015 -0400
@@ -0,0 +1,21 @@
+diff -r 717f4d34f468 io.c
+--- io.c	Tue Mar 05 06:12:37 2013 +0000
++++ io.c	Tue Mar 05 06:17:04 2013 +0000
+@@ -8020,7 +8020,7 @@
+ 
+ #define DEFULT_IOCTL_NARG_LEN (256)
+ 
+-#ifdef __linux__
++#if defined(__GLIBC__) && defined(__linux__)
+ static long
+ linux_iocparm_len(ioctl_req_t cmd)
+ {
+@@ -8053,7 +8053,7 @@
+ #endif
+ #ifdef IOCPARM_LEN
+     len = IOCPARM_LEN(cmd);	/* on BSDish systems we're safe */
+-#elif defined(__linux__)
++#elif defined(__GLIBC__) && defined(__linux__)
+     len = linux_iocparm_len(cmd);
+ #else
+     /* otherwise guess at what's safe */
