# HG changeset patch
# Parent ab20031e6af8a77b47ac5baad57e07a2d4d47a12

diff -r ab20031e6af8 lang/gawk/Makefile
--- a/lang/gawk/Makefile	Tue Oct 09 17:59:17 2012 -0400
+++ b/lang/gawk/Makefile	Tue Oct 09 17:59:22 2012 -0400
@@ -38,6 +38,9 @@
 
 INSTALLATION_DIRS+=	gnu/bin gnu/man/man1
 
+# [Gregor] Poisoning this build since it breaks things
+CONFIGURE_ARGS+=	--help
+
 post-install:
 	${LN} -s ../../bin/gawk ${DESTDIR}${PREFIX}/gnu/bin/awk
 	${LN} -s ../../../man/man1/gawk.1 ${DESTDIR}${PREFIX}/gnu/man/man1/awk.1
