# HG changeset patch
# Parent f3b44044d40c7fd9d0f38df5a00f7296926294a7

diff -r f3b44044d40c lang/perl5/Makefile
--- a/lang/perl5/Makefile	Mon Feb 18 13:22:07 2013 -0500
+++ b/lang/perl5/Makefile	Mon Feb 18 13:23:27 2013 -0500
@@ -46,6 +46,7 @@
 HAS_CONFIGURE=		yes
 CONFIGURE_SCRIPT=	./Configure
 CONFIGURE_ARGS+=	-sde
+CONFIGURE_ARGS+=        -D inc_version_list=none
 MAKE_ENV+=		LC_ALL="C"
 
 TEST_TARGET=		test
@@ -203,6 +204,13 @@
 
 pre-configure:
 	cd ${WRKSRC} && find `pwd` -name "*.orig" -type f -exec ${RM} -f {} \;
+	sed 's/-fstack-protector/-fno-stack-protector/g' ${WRKSRC}/Configure > ${WRKSRC}/Configure.mod
+	mv ${WRKSRC}/Configure.mod ${WRKSRC}/Configure
+	chmod a+x ${WRKSRC}/Configure
+	
+pre-build:	
+	sed 's/define HAS_CRYPT_R/undef HAS_CRYPT_R/g' ${WRKSRC}/config.h > ${WRKSRC}/config.h.mod
+	mv ${WRKSRC}/config.h.mod ${WRKSRC}/config.h
 
 post-build:
 	${SED}	-e "s,@PERL5@,"${PERL5:Q}",g"				\
