# HG changeset patch
# Parent f6189b1e47aa0412929df1375733f3cde83d9c7a

diff -r f6189b1e47aa lang/f2c/distinfo
--- a/lang/f2c/distinfo	Mon Jul 08 19:20:55 2013 -0400
+++ b/lang/f2c/distinfo	Mon Jul 08 19:21:01 2013 -0400
@@ -6,3 +6,4 @@
 SHA1 (patch-aa) = f1224ff17cdc96f083a247dce88b690e8b939eb0
 SHA1 (patch-ad) = 7f204a45891e3632589ea8fb0e4e379a0a792dea
 SHA1 (patch-ak) = 8652a54df300ddd60d9b1a9594d312349b782cf4
+SHA1 (patch-aq) = dd5d4e79bd1d61d7ce152b0934667ba4a1888692
diff -r f6189b1e47aa lang/f2c/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/lang/f2c/patches/patch-aq	Mon Jul 08 19:21:01 2013 -0400
@@ -0,0 +1,13 @@
+diff -r 6bba7eaf719c src/sysdep.c
+--- src/sysdep.c	Sat Apr 28 14:46:46 2012 +0000
++++ src/sysdep.c	Sat Apr 28 14:47:22 2012 +0000
+@@ -83,6 +83,9 @@
+ #include <sys/wait.h>
+ #include <unistd.h>
+ 
++#include <sys/types.h>
++#include <sys/stat.h>
++
+  void
+ #ifdef KR_headers
+ Un_link_all(cdelete)
