# HG changeset patch
# Parent a772c7a9a143bc46337eb7ea2bc7f44546441ce8

diff -r a772c7a9a143 lang/ruby193-base/Makefile
--- a/lang/ruby193-base/Makefile	Mon Feb 18 12:25:31 2013 -0500
+++ b/lang/ruby193-base/Makefile	Mon Feb 18 12:25:36 2013 -0500
@@ -158,6 +158,9 @@
 		${DESTDIR}${PREFIX}/${RUBY_EG}/openssl
 .endfor
 	${RUBY_GENERATE_PLIST}
+	
+# [Gregor] docs fail for unknown reasons
+CONFIGURE_ARGS+=	--disable-install-doc
 
 .include "../../mk/bdb.buildlink3.mk"
 .include "../../converters/libiconv/buildlink3.mk"
diff -r a772c7a9a143 lang/ruby193-base/distinfo
--- a/lang/ruby193-base/distinfo	Mon Feb 18 12:25:31 2013 -0500
+++ b/lang/ruby193-base/distinfo	Mon Feb 18 12:25:36 2013 -0500
@@ -28,3 +28,4 @@
 SHA1 (patch-test_rubygems_test__gem.rb) = bb6946eceed2112a7b3d8df84f59c793bdb73bb0
 SHA1 (patch-tool_rbinstall.rb) = f0497085a09b31145f57028b30e335b09acabcff
 SHA1 (patch-vsnprintf.c) = 7bb4ca07381536717518143ef7803634168ca9fa
+SHA1 (patch-aq) = 254f92d175ad31c3e99acb0e0ad4b6cce44e4540
diff -r a772c7a9a143 lang/ruby193-base/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/lang/ruby193-base/patches/patch-aq	Mon Feb 18 12:25:36 2013 -0500
@@ -0,0 +1,29 @@
+diff -r fdf359c0709a configure
+--- configure	Mon Jul 02 14:51:59 2012 +0000
++++ configure	Mon Jul 02 14:53:13 2012 +0000
+@@ -16017,10 +16017,10 @@
+   '^?ELF'*) :
+     rb_cv_binary_elf=yes ;; #(
+   *) :
+-    rb_cv_binary_elf=no ;;
+-esac
+-else
+-  rb_cv_binary_elf=no
++    rb_cv_binary_elf=yes ;;
++esac
++else
++  rb_cv_binary_elf=yes
+ fi
+ rm -f core conftest.err conftest.$ac_objext \
+     conftest$ac_exeext conftest.$ac_ext
+diff -r fdf359c0709a io.c
+--- io.c	Mon Jul 02 14:51:59 2012 +0000
++++ io.c	Mon Jul 02 14:53:13 2012 +0000
+@@ -8123,6 +8123,7 @@
+     VALUE arg[8];
+ #endif
+ #if SIZEOF_VOIDP == 8 && defined(HAVE___SYSCALL) && SIZEOF_INT != 8 /* mainly *BSD */
++#undef __syscall
+ # define SYSCALL __syscall
+ # define NUM2SYSCALLID(x) NUM2LONG(x)
+ # define RETVAL2NUM(x) LONG2NUM(x)
