# HG changeset patch
# Parent 06f31780c699452c1ba8daad74fb2e243bd0d253

diff -r 06f31780c699 lang/gawk/Makefile
--- a/lang/gawk/Makefile	Mon Feb 18 12:29:50 2013 -0500
+++ b/lang/gawk/Makefile	Mon Feb 18 12:29:55 2013 -0500
@@ -37,6 +37,9 @@
 
 INSTALLATION_DIRS+=	${PKGGNUDIR}bin ${PKGGNUDIR}${PKGMANDIR}/man1
 
+# [Gregor] Poisoning this build since it breaks things
+CONFIGURE_ARGS+=	--help
+
 post-install:
 	${LN} -s ${PREFIX}/bin/gawk ${DESTDIR}${PREFIX}/${PKGGNUDIR}bin/awk
 	${LN} -s ${PREFIX}/${PKGMANDIR}/man1/gawk.1 ${DESTDIR}${PREFIX}/${PKGGNUDIR}${PKGMANDIR}/man1/awk.1
