# HG changeset patch
# Parent cf01c651775c46b6b2afb103601c1d2047eb7c9b
diff -r cf01c651775c -r 1b14d9022630 lang/g95/Makefile
--- a/lang/g95/Makefile	Fri Aug 02 19:29:15 2013 -0400
+++ b/lang/g95/Makefile	Mon Aug 05 19:51:19 2013 -0400
@@ -28,6 +28,10 @@
 GCC_CONFIGURE_ARGS+=	--enable-languages=c
 GCC_CONFIGURE_ARGS+=	--disable-multilib
 
+# [Gregor] No mudflap or libssp on musl
+GCC_CONFIGURE_ARGS+=	--disable-libmudflap --disable-libssp
+LIBS+=			-liconv
+
 .include "../../mk/bsd.prefs.mk"
 .if ${OPSYS} == "SunOS"
 GCC_CONFIGURE_ENV+=	OBJDUMP=/usr/sfw/bin/gobjdump
diff -r cf01c651775c -r 1b14d9022630 lang/g95/distinfo
--- a/lang/g95/distinfo	Fri Aug 02 19:29:15 2013 -0400
+++ b/lang/g95/distinfo	Mon Aug 05 19:51:19 2013 -0400
@@ -26,3 +26,4 @@
 SHA1 (patch-libf95.a-0.93_math_ff.c) = 353121bc08a86765d18b0e6f051ba91f7e65e9c2
 SHA1 (patch-libf95.a-0.93_math_x87.S) = d1eb02bd2358b6abb9b8dc3e32629cba540adcc4
 SHA1 (patch-libf95.a-0.93_quad_power16.c) = fc7375aa90ecf75bb8ba26d41739614040b16e09
+SHA1 (patch-musl) = 8c02beac40b41abfb0e73e83c3bfe4ec3c101a0f
diff -r cf01c651775c -r 1b14d9022630 lang/g95/patches/patch-musl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/lang/g95/patches/patch-musl	Mon Aug 05 19:51:19 2013 -0400
@@ -0,0 +1,121 @@
+diff -r b69a113afeab gcc-4.1.2/config.sub
+--- ../gcc-4.1.2/config.sub	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/config.sub	Sat Aug 03 11:35:47 2013 +0000
+@@ -119,7 +119,7 @@
+ # Here we must recognize all the valid KERNEL-OS combinations.
+ maybe_os=`echo $1 | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\2/'`
+ case $maybe_os in
+-  nto-qnx* | linux-gnu* | linux-dietlibc | linux-uclibc* | uclinux-uclibc* | uclinux-gnu* | \
++  nto-qnx* | linux-gnu* | linux-dietlibc | linux-uclibc* | linux-musl* | uclinux-uclibc* | uclinux-gnu* | \
+   kfreebsd*-gnu* | knetbsd*-gnu* | netbsd*-gnu* | storm-chaos* | os2-emx* | rtmk-nova*)
+     os=-$maybe_os
+     basic_machine=`echo $1 | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\1/'`
+@@ -1195,7 +1195,7 @@
+ 	      | -udi* | -eabi* | -lites* | -ieee* | -go32* | -aux* \
+ 	      | -chorusos* | -chorusrdb* \
+ 	      | -cygwin* | -pe* | -psos* | -moss* | -proelf* | -rtems* \
+-	      | -mingw32* | -linux-gnu* | -linux-uclibc* | -uxpv* | -beos* | -mpeix* | -udk* \
++	      | -mingw32* | -linux-gnu* | -linux-uclibc* | -linux-musl* | -uxpv* | -beos* | -mpeix* | -udk* \
+ 	      | -interix* | -uwin* | -mks* | -rhapsody* | -darwin* | -opened* \
+ 	      | -openstep* | -oskit* | -conix* | -pw32* | -nonstopux* \
+ 	      | -storm-chaos* | -tops10* | -tenex* | -tops20* | -its* \
+diff -r b69a113afeab gcc-4.1.2/gcc/config.gcc
+--- ../gcc-4.1.2/gcc/config.gcc	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/gcc/config.gcc	Sat Aug 03 11:35:47 2013 +0000
+@@ -469,6 +469,14 @@
+     "" | yes | posix) thread_file='posix' ;;
+   esac
+   tmake_file="t-slibgcc-elf-ver t-linux"
++  case ${target} in
++    *-*-*musl*)
++      tm_defines="${tm_defines} MUSL_LIBC=1"
++      ;;
++    *)
++      tm_defines="${tm_defines} MUSL_LIBC=0"
++      ;;
++  esac
+   ;;
+ *-*-gnu*)
+   # On the Hurd, the setup is just about the same on
+diff -r b69a113afeab gcc-4.1.2/gcc/config/i386/linux.h
+--- ../gcc-4.1.2/gcc/config/i386/linux.h	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/gcc/config/i386/linux.h	Sat Aug 03 11:35:47 2013 +0000
+@@ -105,7 +105,11 @@
+ /* If ELF is the default format, we should not use /lib/elf.  */
+ 
+ #define LINK_EMULATION "elf_i386"
++#if !MUSL_LIBC
+ #define DYNAMIC_LINKER "/lib/ld-linux.so.2"
++#else
++#define DYNAMIC_LINKER "/lib/ld-musl-i386.so.1"
++#endif
+ 
+ #undef  SUBTARGET_EXTRA_SPECS
+ #define SUBTARGET_EXTRA_SPECS \
+diff -r b69a113afeab gcc-4.1.2/gcc/config/i386/linux64.h
+--- ../gcc-4.1.2/gcc/config/i386/linux64.h	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/gcc/config/i386/linux64.h	Sat Aug 03 11:35:47 2013 +0000
+@@ -55,6 +55,7 @@
+    done.  */
+ 
+ #undef	LINK_SPEC
++#if !MUSL_LIBC
+ #define LINK_SPEC "%{!m32:-m elf_x86_64} %{m32:-m elf_i386} \
+   %{shared:-shared} \
+   %{!shared: \
+@@ -63,6 +64,16 @@
+       %{m32:%{!dynamic-linker:-dynamic-linker /lib/ld-linux.so.2}} \
+       %{!m32:%{!dynamic-linker:-dynamic-linker /lib64/ld-linux-x86-64.so.2}}} \
+     %{static:-static}}"
++#else
++#define LINK_SPEC "%{!m32:-m elf_x86_64} %{m32:-m elf_i386} \
++  %{shared:-shared} \
++  %{!shared: \
++    %{!static: \
++      %{rdynamic:-export-dynamic} \
++      %{m32:%{!dynamic-linker:-dynamic-linker /lib/ld-musl-i386.so.1}} \
++      %{!m32:%{!dynamic-linker:-dynamic-linker /lib64/ld-musl-x86_64.so.1}}} \
++    %{static:-static}}"
++#endif
+ 
+ /* Similar to standard Linux, but adding -ffast-math support.  */
+ #undef  ENDFILE_SPEC
+diff -r b69a113afeab gcc-4.1.2/gcc/config/mips/linux.h
+--- ../gcc-4.1.2/gcc/config/mips/linux.h	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/gcc/config/mips/linux.h	Sat Aug 03 11:35:47 2013 +0000
+@@ -103,6 +103,8 @@
+ #undef MIPS_DEFAULT_GVALUE
+ #define MIPS_DEFAULT_GVALUE 0
+ 
++#define MUSL_DYNAMIC_LINKER "/lib/ld-musl-mips.so.1"
++
+ /* Borrowed from sparc/linux.h */
+ #undef LINK_SPEC
+ #define LINK_SPEC \
+diff -r b69a113afeab gcc-4.1.2/gcc/ginclude/stddef.h
+--- ../gcc-4.1.2/gcc/ginclude/stddef.h	Sat Aug 03 11:32:59 2013 +0000
++++ ../gcc-4.1.2/gcc/ginclude/stddef.h	Sat Aug 03 11:35:47 2013 +0000
+@@ -193,6 +193,7 @@
+ #ifndef _GCC_SIZE_T
+ #ifndef _SIZET_
+ #ifndef __size_t
++#ifndef __DEFINED_size_t /* musl */
+ #define __size_t__	/* BeOS */
+ #define __SIZE_T__	/* Cray Unicos/Mk */
+ #define _SIZE_T
+@@ -209,6 +210,7 @@
+ #define ___int_size_t_h
+ #define _GCC_SIZE_T
+ #define _SIZET_
++#define __DEFINED_size_t /* musl */
+ #if (defined (__FreeBSD__) && (__FreeBSD__ >= 5)) || defined(__DragonFly__)
+ /* __size_t is a typedef on FreeBSD 5 and on DragonFly BSD!, must not trash it. */
+ #else
+@@ -223,6 +225,7 @@
+ typedef long ssize_t;
+ #endif /* __BEOS__ */
+ #endif /* !(defined (__GNUG__) && defined (size_t)) */
++#endif /* __DEFINED_size_t */
+ #endif /* __size_t */
+ #endif /* _SIZET_ */
+ #endif /* _GCC_SIZE_T */
