# HG changeset patch
# Parent 916b1d0870e2477215d83c622dcdef1f5dc66cbe

diff -r 916b1d0870e2 lang/python27/PLIST.common
--- a/lang/python27/PLIST.common	Tue Oct 09 17:54:08 2012 -0400
+++ b/lang/python27/PLIST.common	Tue Oct 09 17:54:13 2012 -0400
@@ -1836,7 +1836,6 @@
 lib/python${PY_VER_SUFFIX}/lib-dynload/itertools.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/math.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/mmap.so
-${PLIST.nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis.so
 ${PLIST.no-nis}lib/python${PY_VER_SUFFIX}/lib-dynload/nis_failed.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/operator.so
 lib/python${PY_VER_SUFFIX}/lib-dynload/parser.so
