# HG changeset patch
# Parent 74dca1bb2091b51f6486ae114bbb8aefedca35f5

diff -r 74dca1bb2091 math/nickle/distinfo
--- a/math/nickle/distinfo	Mon Apr 08 12:57:25 2013 -0400
+++ b/math/nickle/distinfo	Mon Apr 08 12:57:30 2013 -0400
@@ -4,3 +4,4 @@
 RMD160 (nickle-2.70.tar.gz) = 567028075e57c17a015253d4544a12f140aebb79
 Size (nickle-2.70.tar.gz) = 488081 bytes
 SHA1 (patch-aa) = 0b1e03f3e42ea088c5a81109e9534c77efe30659
+SHA1 (patch-aq) = 6612db40b26cf897cac913b721d2940bdcb7e5ba
diff -r 74dca1bb2091 math/nickle/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/math/nickle/patches/patch-aq	Mon Apr 08 12:57:30 2013 -0400
@@ -0,0 +1,12 @@
+diff -r 32bd16c8749a float.c
+--- float.c	Sat Aug 11 06:46:53 2012 +0000
++++ float.c	Sat Aug 11 06:51:48 2012 +0000
+@@ -1142,7 +1142,7 @@
+     double_digit    dd;
+     if (d == 0.0) RETURN (Zero);
+     e = ilogb (d);
+-    m = significand (d);
++    m = scalb (d, (double) -ilogb(d));
+     ms = Positive;
+     if (m < 0)
+     {
