# HG changeset patch
# Parent 3eaf5ae1534e228f62f9bb2e234f2b8f4942029b

diff -r 3eaf5ae1534e meta-pkgs/xfce4/Makefile
--- a/meta-pkgs/xfce4/Makefile	Mon Feb 18 12:25:36 2013 -0500
+++ b/meta-pkgs/xfce4/Makefile	Mon Feb 18 12:25:41 2013 -0500
@@ -16,7 +16,8 @@
 .include "../../print/xfce4-print/buildlink3.mk"
 .include "../../sysutils/xfce4-appfinder/buildlink3.mk"
 .include "../../sysutils/xfce4-thunar/buildlink3.mk"
-.include "../../sysutils/xfce4-volman/buildlink3.mk"
+# [Gregor] volman depends on hal
+#.include "../../sysutils/xfce4-volman/buildlink3.mk"
 .include "../../sysutils/xfce4-xarchiver/buildlink3.mk"
 .include "../../time/xfce4-orage/buildlink3.mk"
 .include "../../wm/xfce4-wm-themes/buildlink3.mk"
