# HG changeset patch
# Parent 198ba5f00887ec3598a4c98bae90a40a5b936893

diff -r 198ba5f00887 multimedia/libvpx/Makefile
--- a/multimedia/libvpx/Makefile	Sat Mar 14 00:08:30 2015 -0400
+++ b/multimedia/libvpx/Makefile	Sat Mar 14 00:08:36 2015 -0400
@@ -86,6 +86,9 @@
 .  endif
 .endif
 
+# [Gregor] bash script
+BUILD_DEPENDS+=bash-[0-9]*:../../shells/bash
+
 .include "../../mk/compiler.mk"
 .if !empty(PKGSRC_COMPILER:Mclang)
 CXXFLAGS+=	-std=c++11
