# HG changeset patch
# Parent c3901a52d87c1e7c92777c2126903dcf3133b0b2
diff --git a/multimedia/x264-devel/distinfo b/multimedia/x264-devel/distinfo
--- a/multimedia/x264-devel/distinfo
+++ b/multimedia/x264-devel/distinfo
@@ -8,3 +8,4 @@
 SHA1 (patch-ac) = a1bd65058559ec1704e2b64dade9f5a84066e67a
 SHA1 (patch-ak) = 3c8d6650147ad6a0043d705464267a5d947da488
 SHA1 (patch-ba) = 074eaf75ac7b3e8019950360283e1c65dc8a8109
+SHA1 (patch-aq) = 4bd298937c9662c35212e405192f841629340638
diff --git a/multimedia/x264-devel/patches/patch-aq b/multimedia/x264-devel/patches/patch-aq
new file mode 100644
--- /dev/null
+++ b/multimedia/x264-devel/patches/patch-aq
@@ -0,0 +1,21 @@
+diff -r 95a6fda9df62 common/cpu.c
+--- common/cpu.c	Wed Aug 15 18:41:02 2012 +0000
++++ common/cpu.c	Wed Aug 15 18:47:32 2012 +0000
+@@ -378,6 +378,7 @@
+     return sysconf( _SC_NPROCESSORS_ONLN );
+ 
+ #elif SYS_LINUX
++#if 0
+     cpu_set_t p_aff;
+     memset( &p_aff, 0, sizeof(p_aff) );
+     if( sched_getaffinity( 0, sizeof(p_aff), &p_aff ) )
+@@ -390,6 +391,9 @@
+         np += (((uint8_t *)&p_aff)[bit / 8] >> (bit % 8)) & 1;
+     return np;
+ #endif
++#else
++    return 1;
++#endif
+ 
+ #elif SYS_BEOS
+     system_info info;
