# HG changeset patch
# Parent e979cf1eabba0c879f91e06c7e100763e3f951f8

diff -r e979cf1eabba multimedia/x264-devel/distinfo
--- a/multimedia/x264-devel/distinfo	Mon Jul 08 19:25:13 2013 -0400
+++ b/multimedia/x264-devel/distinfo	Mon Jul 08 19:25:20 2013 -0400
@@ -8,3 +8,4 @@
 SHA1 (patch-ac) = 42aaa1f91cd87923442d23ff222e1b37ed43948b
 SHA1 (patch-ak) = bb73c847d6f7be5cee2eee1d77060b5633e5dfc2
 SHA1 (patch-ba) = a8f12131e23953704361fb499b568f8f233c6be6
+SHA1 (patch-aq) = 4bd298937c9662c35212e405192f841629340638
diff -r e979cf1eabba multimedia/x264-devel/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/multimedia/x264-devel/patches/patch-aq	Mon Jul 08 19:25:20 2013 -0400
@@ -0,0 +1,21 @@
+diff -r 95a6fda9df62 common/cpu.c
+--- common/cpu.c	Wed Aug 15 18:41:02 2012 +0000
++++ common/cpu.c	Wed Aug 15 18:47:32 2012 +0000
+@@ -378,6 +378,7 @@
+     return sysconf( _SC_NPROCESSORS_ONLN );
+ 
+ #elif SYS_LINUX
++#if 0
+     cpu_set_t p_aff;
+     memset( &p_aff, 0, sizeof(p_aff) );
+     if( sched_getaffinity( 0, sizeof(p_aff), &p_aff ) )
+@@ -390,6 +391,9 @@
+         np += (((uint8_t *)&p_aff)[bit / 8] >> (bit % 8)) & 1;
+     return np;
+ #endif
++#else
++    return 1;
++#endif
+ 
+ #elif SYS_BEOS
+     system_info info;
