# HG changeset patch
# Parent 0385d2acf6de1093bd1826a5f09c052255e8e1d3

diff -r 0385d2acf6de multimedia/libvpx/Makefile
--- a/multimedia/libvpx/Makefile	Mon Jul 08 19:27:05 2013 -0400
+++ b/multimedia/libvpx/Makefile	Mon Jul 08 19:27:11 2013 -0400
@@ -74,5 +74,8 @@
 post-extract:
 	${CP} ${FILESDIR}/strip_fPIC.sh ${WRKSRC}
 
+# [Gregor] bash script
+BUILD_DEPENDS+=bash-[0-9]*:../../shells/bash
+
 .include "../../mk/pthread.buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
