# HG changeset patch
# Parent 2d6bc0a8d169e3c124dc24d12b92c54db12ed5ff
diff -r 2d6bc0a8d169 -r 5ad5e6a228e8 multimedia/libvpx/Makefile
--- a/multimedia/libvpx/Makefile	Fri Apr 05 19:17:25 2013 -0400
+++ b/multimedia/libvpx/Makefile	Fri Apr 05 19:18:01 2013 -0400
@@ -74,5 +74,8 @@
 post-extract:
 	${CP} ${FILESDIR}/strip_fPIC.sh ${WRKSRC}
 
+# [Gregor] bash script
+BUILD_DEPENDS+=bash-[0-9]*:../../shells/bash
+
 .include "../../mk/pthread.buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
