diff -r 6c246edf2d0a net/mDNSResponder/distinfo
--- a/net/mDNSResponder/distinfo	Sat Mar 14 00:07:42 2015 -0400
+++ b/net/mDNSResponder/distinfo	Sat Mar 14 00:07:51 2015 -0400
@@ -10,6 +10,7 @@
 SHA1 (patch-ad) = 612390897ad2f395ceb7394b644f977c4858d6cc
 SHA1 (patch-ae) = 49091d627fd3a800ed027566c728022b901d126b
 SHA1 (patch-af) = bce933daa008f081b72c5d509e84704be13360dc
+SHA1 (patch-qq) = 8afa47acb539c5ace46182156c61812ea8ab216e
 SHA1 (patch-mDNSShared_dnsextd__parser.y) = dde7eca76c2b5158c2ac96764f91ce773464094a
 SHA1 (patch-mDNSShared_dnssd__clientstub.c) = 591801ae08dce4a44c06dd52a46fcd63807eafcc
 SHA1 (patch-mDNSShared_uds__daemon.c) = df279d3871bb6ead1563c79c99c4d2dc6d91ff79
diff -r 6c246edf2d0a net/mDNSResponder/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/net/mDNSResponder/patches/patch-qq	Sat Mar 14 00:07:51 2015 -0400
@@ -0,0 +1,51 @@
+diff -r 807537ed80b4 mDNSPosix/nss_mdns.c
+--- mDNSPosix/nss_mdns.c	Tue Feb 19 05:06:36 2013 +0000
++++ mDNSPosix/nss_mdns.c	Tue Feb 19 05:14:07 2013 +0000
+@@ -379,14 +379,22 @@
+ #define ENTNAME  hostent
+ #define DATABASE "hosts"
+ 
+-#include <nss.h>
++//#include <nss.h>
+ 	// For nss_status
+ #include <netdb.h>
+ 	// For hostent
+ #include <sys/types.h>
+ 	// For size_t
+ 
+-typedef enum nss_status nss_status;
++#define NSS_STATUS_SUCCESS 0
++#define NSS_STATUS_NOTFOUND 1
++#define NSS_STATUS_UNAVAIL 2
++#define NSS_STATUS_TRYAGAIN 3
++#define NSS_STATUS_RETURN 4
++#define NETDB_INTERNAL NULL
++
++//typedef enum nss_status nss_status;
++typedef int nss_status;
+ typedef struct hostent hostent;
+ 
+ /*
+diff -r 807537ed80b4 mDNSShared/CommonServices.h
+--- mDNSShared/CommonServices.h	Tue Feb 19 05:06:36 2013 +0000
++++ mDNSShared/CommonServices.h	Tue Feb 19 05:14:07 2013 +0000
+@@ -211,6 +211,7 @@
+ 	
+ 	#include	<stdint.h>
+ 	#include	<arpa/inet.h>
++	#include	<sys/socket.h>
+ 	
+ #elif ( TARGET_OS_NETBSD )
+ 
+diff -r 807537ed80b4 mDNSShared/dnssd_clientstub.c
+--- mDNSShared/dnssd_clientstub.c	Tue Feb 19 05:06:36 2013 +0000
++++ mDNSShared/dnssd_clientstub.c	Tue Feb 19 05:14:07 2013 +0000
+@@ -71,7 +71,7 @@
+ 		}
+ #else
+ 
+-	#include <sys/fcntl.h>		// For O_RDWR etc.
++	#include <fcntl.h>		// For O_RDWR etc.
+ 	#include <sys/time.h>
+ 	#include <sys/socket.h>
+ 	#include <syslog.h>
