# HG changeset patch
# Parent 0f4fadda884a2e7b9a13337614b00a05608610b8

diff -r 0f4fadda884a net/libfetch/files/ftp.c
--- a/net/libfetch/files/ftp.c	Mon Apr 08 12:53:54 2013 -0400
+++ b/net/libfetch/files/ftp.c	Mon Apr 08 12:53:58 2013 -0400
@@ -59,8 +59,10 @@
 
 #ifdef __linux__
 /* Keep this down to Linux, it can create surprises else where. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 #if HAVE_CONFIG_H
 #include "config.h"
diff -r 0f4fadda884a net/libfetch/files/http.c
--- a/net/libfetch/files/http.c	Mon Apr 08 12:53:54 2013 -0400
+++ b/net/libfetch/files/http.c	Mon Apr 08 12:53:58 2013 -0400
@@ -65,8 +65,10 @@
 
 #if defined(__linux__) || defined(__MINT__)
 /* Keep this down to Linux or MiNT, it can create surprises elsewhere. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 /* Needed for gmtime_r on Interix */
 #define _REENTRANT
