# HG changeset patch
# Parent 21e0dcd4422aa8b0b94ff9c80f26c9f26c96a2df

diff -r 21e0dcd4422a net/libfetch/files/ftp.c
--- a/net/libfetch/files/ftp.c	Mon Jul 08 19:21:18 2013 -0400
+++ b/net/libfetch/files/ftp.c	Mon Jul 08 19:21:23 2013 -0400
@@ -59,8 +59,10 @@
 
 #ifdef __linux__
 /* Keep this down to Linux, it can create surprises else where. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 #if HAVE_CONFIG_H
 #include "config.h"
diff -r 21e0dcd4422a net/libfetch/files/http.c
--- a/net/libfetch/files/http.c	Mon Jul 08 19:21:18 2013 -0400
+++ b/net/libfetch/files/http.c	Mon Jul 08 19:21:23 2013 -0400
@@ -65,8 +65,10 @@
 
 #if defined(__linux__) || defined(__MINT__)
 /* Keep this down to Linux or MiNT, it can create surprises elsewhere. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 /* Needed for gmtime_r on Interix */
 #define _REENTRANT
