# HG changeset patch
# Parent 3f91c5c5c3e2566c43b2f2b206ea250f3524ea69

diff -r 3f91c5c5c3e2 net/tightvnc/Makefile
--- a/net/tightvnc/Makefile	Mon Feb 18 13:23:32 2013 -0500
+++ b/net/tightvnc/Makefile	Mon Feb 18 13:24:31 2013 -0500
@@ -27,6 +27,7 @@
 .if !empty(MACHINE_PLATFORM:MNetBSD-*-x86_64)
 _WRAP_EXTRA_ARGS.CC+=	-DDDXTIME -D_XSERVER64 -DGCCUSESGAS
 .endif
+_WRAP_EXTRA_ARGS.CC+=	-DNOFILES_MAX=256
 
 .if ${X11_TYPE} == "modular"
 DEPENDS+=		rgb>=1.0.1:../../x11/rgb
@@ -55,6 +56,13 @@
 
 INSTALLATION_DIRS=	bin ${PKGMANDIR}/man1 share/examples/tightvnc
 
+pre-build:
+	echo '#define DEFAULT_CPP "cpp"' >> ${WRKSRC}/Xvnc/config/imake/imakemdep.h
+	sed 's/alloca/malloc/g ; s/-x c -/-x c - -x none/' ${WRKSRC}/Xvnc/config/imake/imake.c > ${WRKSRC}/Xvnc/config/imake/imake.c.mod
+	mv ${WRKSRC}/Xvnc/config/imake/imake.c.mod ${WRKSRC}/Xvnc/config/imake/imake.c
+	sed 's/HasFfsl[ \t]*YES/HasFfsl NO/' ${WRKSRC}/Xvnc/config/cf/linux.cf > ${WRKSRC}/Xvnc/config/cf/linux.cf.mod
+	mv ${WRKSRC}/Xvnc/config/cf/linux.cf.mod ${WRKSRC}/Xvnc/config/cf/linux.cf
+
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/Xvnc/programs/Xserver/Xvnc \
 	    ${DESTDIR}${PREFIX}/bin/Xvnc
