# HG changeset patch
# Parent 1f35adb2e51e4b156c5871276ea54a3df919978f

diff -r 1f35adb2e51e net/libfetch/files/ftp.c
--- a/net/libfetch/files/ftp.c	Mon Feb 18 12:25:47 2013 -0500
+++ b/net/libfetch/files/ftp.c	Mon Feb 18 12:25:52 2013 -0500
@@ -59,8 +59,10 @@
 
 #ifdef __linux__
 /* Keep this down to Linux, it can create surprises else where. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 #if HAVE_CONFIG_H
 #include "config.h"
diff -r 1f35adb2e51e net/libfetch/files/http.c
--- a/net/libfetch/files/http.c	Mon Feb 18 12:25:47 2013 -0500
+++ b/net/libfetch/files/http.c	Mon Feb 18 12:25:52 2013 -0500
@@ -65,8 +65,10 @@
 
 #if defined(__linux__) || defined(__MINT__)
 /* Keep this down to Linux or MiNT, it can create surprises elsewhere. */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
+#endif
 
 /* Needed for gmtime_r on Interix */
 #define _REENTRANT
