# HG changeset patch
# Parent 12ad41b3c66846f0d20cd4ef2ae234644feea3d7

diff -r 12ad41b3c668 pkgtools/libnbcompat/files/lchflags.c
--- a/pkgtools/libnbcompat/files/lchflags.c	Mon Apr 08 12:54:09 2013 -0400
+++ b/pkgtools/libnbcompat/files/lchflags.c	Mon Apr 08 12:54:13 2013 -0400
@@ -39,6 +39,7 @@
 int
 lchflags(const char *path, unsigned long flags)
 {
+#if 0
 	struct stat psb;
 
 	if (lstat(path, &psb) == -1)
@@ -47,4 +48,6 @@
 		return 0;
 	}
 	return (chflags(path, flags));
+#endif
+	return -1;
 }
diff -r 12ad41b3c668 pkgtools/libnbcompat/files/unvis.c
--- a/pkgtools/libnbcompat/files/unvis.c	Mon Apr 08 12:54:09 2013 -0400
+++ b/pkgtools/libnbcompat/files/unvis.c	Mon Apr 08 12:54:13 2013 -0400
@@ -83,6 +83,8 @@
 #define	isoctal(c)	(((unsigned char)(c)) >= '0' && ((unsigned char)(c)) <= '7')
 #define xtod(c)		(isdigit(c) ? (c - '0') : ((tolower(c) - 'a') + 10))
 
+int __nbcompat_unvis13(char *cp, int c, int *astate, int flag);
+
 int
 unvis(cp, c, astate, flag)
 	char *cp;
