# HG changeset patch
# Parent 6c92029da189c4bbea442bfcbb0a2c037f901d39

diff -r 6c92029da189 pkgtools/pkg_install-info/files/install-info.c
--- a/pkgtools/pkg_install-info/files/install-info.c	Mon Apr 08 12:54:14 2013 -0400
+++ b/pkgtools/pkg_install-info/files/install-info.c	Mon Apr 08 12:54:18 2013 -0400
@@ -21,6 +21,8 @@
 #include "system.h"
 #include <getopt.h>
 
+#include <stdlib.h>
+
 static char *progname = "install-info";
 
 int menu_line_equal (), menu_line_lessp ();
diff -r 6c92029da189 pkgtools/pkg_install-info/files/system.h
--- a/pkgtools/pkg_install-info/files/system.h	Mon Apr 08 12:54:14 2013 -0400
+++ b/pkgtools/pkg_install-info/files/system.h	Mon Apr 08 12:54:18 2013 -0400
@@ -21,7 +21,9 @@
 #ifndef TEXINFO_SYSTEM_H
 #define TEXINFO_SYSTEM_H
 
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
 
 #include <config.h>
 
diff -r 6c92029da189 pkgtools/pkg_install-info/files/xexit.c
--- a/pkgtools/pkg_install-info/files/xexit.c	Mon Apr 08 12:54:14 2013 -0400
+++ b/pkgtools/pkg_install-info/files/xexit.c	Mon Apr 08 12:54:18 2013 -0400
@@ -19,6 +19,8 @@
 
 #include "system.h"
 
+#include <stdlib.h>
+
 /* SunOS 4.1.1 gets STDC_HEADERS defined, but it doesn't provide
    EXIT_FAILURE.  So far no system has defined one of EXIT_FAILURE and
    EXIT_SUCCESS without the other.  */
