# HG changeset patch
# Parent ce7d7ec7f7f382f8e524ebf68d872aed2448abe6

diff -r ce7d7ec7f7f3 pkgtools/libnbcompat/files/lchflags.c
--- a/pkgtools/libnbcompat/files/lchflags.c	Tue Oct 09 17:54:56 2012 -0400
+++ b/pkgtools/libnbcompat/files/lchflags.c	Tue Oct 09 17:55:02 2012 -0400
@@ -39,6 +39,7 @@
 int
 lchflags(const char *path, unsigned long flags)
 {
+#if 0
 	struct stat psb;
 
 	if (lstat(path, &psb) == -1)
@@ -47,4 +48,6 @@
 		return 0;
 	}
 	return (chflags(path, flags));
+#endif
+	return -1;
 }
diff -r ce7d7ec7f7f3 pkgtools/libnbcompat/files/unvis.c
--- a/pkgtools/libnbcompat/files/unvis.c	Tue Oct 09 17:54:56 2012 -0400
+++ b/pkgtools/libnbcompat/files/unvis.c	Tue Oct 09 17:55:02 2012 -0400
@@ -83,6 +83,8 @@
 #define	isoctal(c)	(((unsigned char)(c)) >= '0' && ((unsigned char)(c)) <= '7')
 #define xtod(c)		(isdigit(c) ? (c - '0') : ((tolower(c) - 'a') + 10))
 
+int __nbcompat_unvis13(char *cp, int c, int *astate, int flag);
+
 int
 unvis(cp, c, astate, flag)
 	char *cp;
