# HG changeset patch
# Parent 3d365a76a0a351c1434212a5fb1f4a71b9834f3d

diff -r 3d365a76a0a3 pkgtools/libnbcompat/files/lchflags.c
--- a/pkgtools/libnbcompat/files/lchflags.c	Mon Feb 18 12:26:08 2013 -0500
+++ b/pkgtools/libnbcompat/files/lchflags.c	Mon Feb 18 12:26:12 2013 -0500
@@ -39,6 +39,7 @@
 int
 lchflags(const char *path, unsigned long flags)
 {
+#if 0
 	struct stat psb;
 
 	if (lstat(path, &psb) == -1)
@@ -47,4 +48,6 @@
 		return 0;
 	}
 	return (chflags(path, flags));
+#endif
+	return -1;
 }
diff -r 3d365a76a0a3 pkgtools/libnbcompat/files/unvis.c
--- a/pkgtools/libnbcompat/files/unvis.c	Mon Feb 18 12:26:08 2013 -0500
+++ b/pkgtools/libnbcompat/files/unvis.c	Mon Feb 18 12:26:12 2013 -0500
@@ -83,6 +83,8 @@
 #define	isoctal(c)	(((unsigned char)(c)) >= '0' && ((unsigned char)(c)) <= '7')
 #define xtod(c)		(isdigit(c) ? (c - '0') : ((tolower(c) - 'a') + 10))
 
+int __nbcompat_unvis13(char *cp, int c, int *astate, int flag);
+
 int
 unvis(cp, c, astate, flag)
 	char *cp;
