# HG changeset patch
# Parent 343f5b41e6d7affb4dc9987334e6527d36192c8b

diff -r 343f5b41e6d7 pkgtools/pkg_install-info/files/install-info.c
--- a/pkgtools/pkg_install-info/files/install-info.c	Mon Feb 18 12:26:13 2013 -0500
+++ b/pkgtools/pkg_install-info/files/install-info.c	Mon Feb 18 12:26:17 2013 -0500
@@ -21,6 +21,8 @@
 #include "system.h"
 #include <getopt.h>
 
+#include <stdlib.h>
+
 static char *progname = "install-info";
 
 int menu_line_equal (), menu_line_lessp ();
diff -r 343f5b41e6d7 pkgtools/pkg_install-info/files/system.h
--- a/pkgtools/pkg_install-info/files/system.h	Mon Feb 18 12:26:13 2013 -0500
+++ b/pkgtools/pkg_install-info/files/system.h	Mon Feb 18 12:26:17 2013 -0500
@@ -21,7 +21,9 @@
 #ifndef TEXINFO_SYSTEM_H
 #define TEXINFO_SYSTEM_H
 
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
 
 #include <config.h>
 
diff -r 343f5b41e6d7 pkgtools/pkg_install-info/files/xexit.c
--- a/pkgtools/pkg_install-info/files/xexit.c	Mon Feb 18 12:26:13 2013 -0500
+++ b/pkgtools/pkg_install-info/files/xexit.c	Mon Feb 18 12:26:17 2013 -0500
@@ -19,6 +19,8 @@
 
 #include "system.h"
 
+#include <stdlib.h>
+
 /* SunOS 4.1.1 gets STDC_HEADERS defined, but it doesn't provide
    EXIT_FAILURE.  So far no system has defined one of EXIT_FAILURE and
    EXIT_SUCCESS without the other.  */
