# HG changeset patch
# Parent 11a1bb03d3598b09676154bd4b2202df8b868c8f

diff -r 11a1bb03d359 pkgtools/pkg_install/files/add/Makefile.in
--- a/pkgtools/pkg_install/files/add/Makefile.in	Mon Feb 18 12:26:18 2013 -0500
+++ b/pkgtools/pkg_install/files/add/Makefile.in	Mon Feb 18 12:26:22 2013 -0500
@@ -22,7 +22,7 @@
 
 LIBS=		-linstall -lfetch
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lssl -lcrypto
+LIBS+=		-lssl -lcrypto -lz
 .endif
 LIBS+=		@LIBS@
 
diff -r 11a1bb03d359 pkgtools/pkg_install/files/admin/Makefile.in
--- a/pkgtools/pkg_install/files/admin/Makefile.in	Mon Feb 18 12:26:18 2013 -0500
+++ b/pkgtools/pkg_install/files/admin/Makefile.in	Mon Feb 18 12:26:22 2013 -0500
@@ -30,7 +30,7 @@
 .if empty(BOOTSTRAP)
 LIBS=		-linstall -lfetch
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lssl -lcrypto
+LIBS+=		-lssl -lcrypto -lz
 CFLAGS+=	-DHAVE_SSL
 .endif
 LIBS+=		@LIBS@
diff -r 11a1bb03d359 pkgtools/pkg_install/files/info/Makefile.in
--- a/pkgtools/pkg_install/files/info/Makefile.in	Mon Feb 18 12:26:18 2013 -0500
+++ b/pkgtools/pkg_install/files/info/Makefile.in	Mon Feb 18 12:26:22 2013 -0500
@@ -19,7 +19,7 @@
 .if empty(BOOTSTRAP)
 LIBS=		-linstall -lfetch @LIBS@
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lssl -lcrypto
+LIBS+=		-lssl -lcrypto -lz
 .endif
 CPPFLAGS=	@CPPFLAGS@ -I. -I$(srcdir) -I../lib
 .else
diff -r 11a1bb03d359 pkgtools/pkg_install/files/lib/dewey.c
--- a/pkgtools/pkg_install/files/lib/dewey.c	Mon Feb 18 12:26:18 2013 -0500
+++ b/pkgtools/pkg_install/files/lib/dewey.c	Mon Feb 18 12:26:22 2013 -0500
@@ -39,6 +39,7 @@
 #if HAVE_STDLIB_H
 #include <stdlib.h>
 #endif
+#include <unistd.h> /* for ssize_t */
 
 #include "defs.h"
 #include "dewey.h"
