# HG changeset patch
# Parent 174cd7a02853076ec83969f4725eb247f1db844f

diff -r 174cd7a02853 pkgtools/pkg_install/files/add/Makefile.in
--- a/pkgtools/pkg_install/files/add/Makefile.in	Tue Oct 09 17:55:08 2012 -0400
+++ b/pkgtools/pkg_install/files/add/Makefile.in	Tue Oct 09 17:55:13 2012 -0400
@@ -22,7 +22,7 @@
 
 LIBS=		-linstall -lfetch
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lcrypto
+LIBS+=		-lcrypto -lz
 .endif
 LIBS+=		@LIBS@
 
diff -r 174cd7a02853 pkgtools/pkg_install/files/admin/Makefile.in
--- a/pkgtools/pkg_install/files/admin/Makefile.in	Tue Oct 09 17:55:08 2012 -0400
+++ b/pkgtools/pkg_install/files/admin/Makefile.in	Tue Oct 09 17:55:13 2012 -0400
@@ -30,7 +30,7 @@
 .if empty(BOOTSTRAP)
 LIBS=		-linstall -lfetch
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lcrypto
+LIBS+=		-lcrypto -lz
 CFLAGS+=	-DHAVE_SSL
 .endif
 LIBS+=		@LIBS@
diff -r 174cd7a02853 pkgtools/pkg_install/files/info/Makefile.in
--- a/pkgtools/pkg_install/files/info/Makefile.in	Tue Oct 09 17:55:08 2012 -0400
+++ b/pkgtools/pkg_install/files/info/Makefile.in	Tue Oct 09 17:55:13 2012 -0400
@@ -19,7 +19,7 @@
 .if empty(BOOTSTRAP)
 LIBS=		-linstall -lfetch @LIBS@
 .if !empty(SSL_SUPPORT)
-LIBS+=		-lcrypto
+LIBS+=		-lcrypto -lz
 .endif
 CPPFLAGS=	@CPPFLAGS@ -I. -I$(srcdir) -I../lib
 .else
diff -r 174cd7a02853 pkgtools/pkg_install/files/lib/dewey.c
--- a/pkgtools/pkg_install/files/lib/dewey.c	Tue Oct 09 17:55:08 2012 -0400
+++ b/pkgtools/pkg_install/files/lib/dewey.c	Tue Oct 09 17:55:13 2012 -0400
@@ -39,6 +39,7 @@
 #if HAVE_STDLIB_H
 #include <stdlib.h>
 #endif
+#include <unistd.h> /* for ssize_t */
 
 #include "defs.h"
 #include "dewey.h"
