# HG changeset patch
# Parent b4596252c783cf6ef48834efea47f1a8a21a834d

diff -r b4596252c783 print/cups/Makefile
--- a/print/cups/Makefile	Mon Apr 08 12:58:03 2013 -0400
+++ b/print/cups/Makefile	Mon Apr 08 12:58:08 2013 -0400
@@ -71,8 +71,12 @@
 
 .if ${OPSYS} == "SunOS"
 CONFIGURE_ENV+=		LIBS_SSP=-lssp_nonshared
-LIBS.SunOS+=		-liconv
 .endif
+# [Gregor] Seems to be necessary on musl too
+LIBS+=		-liconv
+
+# [Gregor] Needs strl* functions, but excludes them
+CFLAGS+=	-D_BSD_SOURCE
 
 PLIST_VARS+=		notmac
 .if ${OPSYS} != "Darwin"
