# HG changeset patch
# Parent 16620ea08561813ba036a685df680c10a6c4122b

diff -r 16620ea08561 security/policykit/distinfo
--- a/security/policykit/distinfo	Mon Jul 08 19:22:11 2013 -0400
+++ b/security/policykit/distinfo	Mon Jul 08 19:22:17 2013 -0400
@@ -19,3 +19,4 @@
 SHA1 (patch-an) = a0a54235fc72a52ace72a0d04df6c1383f7c583c
 SHA1 (patch-ao) = 0f9a87653f4902285c7420f6269e302eca896a1b
 SHA1 (patch-authdb) = 449b8ac50f81cc0234e17e64bc9892ade7119028
+SHA1 (patch-aq) = 07d7453aba399fd3b61e05a46ec20f6080be7fde
diff -r 16620ea08561 security/policykit/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/security/policykit/patches/patch-aq	Mon Jul 08 19:22:17 2013 -0400
@@ -0,0 +1,11 @@
+diff -r 1ceaf4291bd1 src/polkit-dbus/polkit-resolve-exe-helper.c
+--- src/polkit-dbus/polkit-resolve-exe-helper.c	Sun May 06 03:39:34 2012 +0000
++++ src/polkit-dbus/polkit-resolve-exe-helper.c	Sun May 06 03:39:59 2012 +0000
+@@ -52,6 +52,7 @@
+ #include <utime.h>
+ #include <fcntl.h>
+ #include <dirent.h>
++#include <limits.h>
+ 
+ #include <polkit-dbus/polkit-dbus.h>
+ #include <polkit/polkit-private.h>
