# HG changeset patch
# Parent aa988c695ecb2472bedaaf1beb3506c0a8bc77a3

diff -r aa988c695ecb security/openssl/Makefile
--- a/security/openssl/Makefile	Mon Jul 08 19:22:00 2013 -0400
+++ b/security/openssl/Makefile	Mon Jul 08 19:22:05 2013 -0400
@@ -45,6 +45,7 @@
 # back to ./config's autodetection if not.
 .  if defined(OPENSSL_MACHINE_ARCH) && !empty(OPENSSL_MACHINE_ARCH)
 CONFIGURE_SCRIPT=	./Configure
+CONFIGURE_ARGS+=	no-shared no-dso zlib enable-md2 -Wa,--noexecstack
 .    if !empty(CC_VERSION:Mgcc*)
 CONFIGURE_ARGS+=	solaris${${ABI}==64:?64:}-${OPENSSL_MACHINE_ARCH}-gcc
 .    else
@@ -138,8 +139,19 @@
 
 INSTALLATION_DIRS+=	share/examples/openssl
 
+pre-build:
+	touch ${WRKSRC}/include/memory.h
+	cd ${WRKSRC} && $(MAKE) depend
+
 # Fix the path to perl in various scripts.
 pre-configure:
 	cd ${WRKSRC} && ${PERL5} util/perlpath.pl ${PERL5}
-
+	sed 's/-DTERMIO/-DTERMIOS/g' ${WRKSRC}/Configure > ${WRKSRC}/Configure.mod
+	mv ${WRKSRC}/Configure.mod ${WRKSRC}/Configure
+	chmod a+x ${WRKSRC}/Configure
+	sed 's/defined(linux)/0/' ${WRKSRC}/crypto/ui/ui_openssl.c > ${WRKSRC}/crypto/ui/ui_openssl.c.mod
+	mv ${WRKSRC}/crypto/ui/ui_openssl.c.mod ${WRKSRC}/crypto/ui/ui_openssl.c
+	sed 's/\!defined(_UNICOS)/0/' ${WRKSRC}/apps/speed.c > ${WRKSRC}/apps/speed.c.mod
+	mv ${WRKSRC}/apps/speed.c.mod ${WRKSRC}/apps/speed.c
+	
 .include "../../mk/bsd.pkg.mk"
