# HG changeset patch
# Parent 8957d324dc165ba607650118fad1399a0db01f0c

diff -r 8957d324dc16 security/openssl/Makefile
--- a/security/openssl/Makefile	Mon Feb 18 12:26:39 2013 -0500
+++ b/security/openssl/Makefile	Mon Feb 18 12:26:43 2013 -0500
@@ -64,6 +64,7 @@
 # back to ./config's autodetection if not.
 .  if defined(OPENSSL_MACHINE_ARCH) && !empty(OPENSSL_MACHINE_ARCH)
 CONFIGURE_SCRIPT=	./Configure
+CONFIGURE_ARGS+=	no-shared no-dso zlib enable-md2 -Wa,--noexecstack
 .    if !empty(CC_VERSION:Mgcc*)
 CONFIGURE_ARGS+=	solaris${${ABI}==64:?64:}-${OPENSSL_MACHINE_ARCH}-gcc
 .    else
@@ -157,8 +158,16 @@
 
 INSTALLATION_DIRS+=	share/examples/openssl
 
+pre-build:
+	touch ${WRKSRC}/include/memory.h
+	cd ${WRKSRC} && $(MAKE) depend
+
 # Fix the path to perl in various scripts.
 pre-configure:
 	cd ${WRKSRC} && ${PERL5} util/perlpath.pl ${PERL5}
-
+	sed -i '/^"linux-x86_64/s/-DTERMIO/-DTERMIOS/' ${WRKSRC}/Configure
+	sed -i '/^"linux-elf/s/-DTERMIO/-DTERMIOS/' ${WRKSRC}/Configure
+	sed -i 's/defined(linux)/0/' ${WRKSRC}/crypto/ui/ui_openssl.c
+	sed -i 's/\!defined(_UNICOS)/0/' ${WRKSRC}/apps/speed.c
+	
 .include "../../mk/bsd.pkg.mk"
