# HG changeset patch
# Parent 6910fb3cfd177102a4b26053e18d593e5a1793b1

diff -r 6910fb3cfd17 security/libgpg-error/PLIST
--- a/security/libgpg-error/PLIST	Mon Jul 08 19:21:49 2013 -0400
+++ b/security/libgpg-error/PLIST	Mon Jul 08 19:21:55 2013 -0400
@@ -8,17 +8,3 @@
 share/common-lisp/source/gpg-error/gpg-error-package.lisp
 share/common-lisp/source/gpg-error/gpg-error.asd
 share/common-lisp/source/gpg-error/gpg-error.lisp
-${PLIST.nls}share/locale/cs/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/da/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/de/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/eo/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/fr/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/it/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/ja/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/nl/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/pl/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/ro/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/sv/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/uk/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/vi/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/zh_CN/LC_MESSAGES/libgpg-error.mo
