# HG changeset patch
# Parent f5fa49e7a10ede7e6c53fbd2484b34ebe681b01d

diff -r f5fa49e7a10e security/openssl/Makefile
--- a/security/openssl/Makefile	Mon Feb 18 13:24:34 2013 -0500
+++ b/security/openssl/Makefile	Mon Feb 18 13:26:39 2013 -0500
@@ -64,6 +64,7 @@
 # back to ./config's autodetection if not.
 .  if defined(OPENSSL_MACHINE_ARCH) && !empty(OPENSSL_MACHINE_ARCH)
 CONFIGURE_SCRIPT=	./Configure
+CONFIGURE_ARGS+=	no-shared no-dso zlib enable-md2 -Wa,--noexecstack
 .    if !empty(CC_VERSION:Mgcc*)
 CONFIGURE_ARGS+=	solaris${${ABI}==64:?64:}-${OPENSSL_MACHINE_ARCH}-gcc
 .    else
@@ -157,8 +158,19 @@
 
 INSTALLATION_DIRS+=	share/examples/openssl
 
+pre-build:
+	touch ${WRKSRC}/include/memory.h
+	cd ${WRKSRC} && $(MAKE) depend
+
 # Fix the path to perl in various scripts.
 pre-configure:
 	cd ${WRKSRC} && ${PERL5} util/perlpath.pl ${PERL5}
-
+	sed '/^"linux-x86_64/s/-DTERMIO/-DTERMIOS/i ; /^"linux-elf/s/-DTERMIO/-DTERMIOS/' ${WRKSRC}/Configure > ${WRKSRC}/Configure.mod
+	mv ${WRKSRC}/Configure.mod ${WRKSRC}/Configure
+	chmod a+x ${WRKSRC}/Configure
+	sed 's/defined(linux)/0/' ${WRKSRC}/crypto/ui/ui_openssl.c > ${WRKSRC}/crypto/ui/ui_openssl.c.mod
+	mv ${WRKSRC}/crypto/ui/ui_openssl.c.mod ${WRKSRC}/crypto/ui/ui_openssl.c
+	sed 's/\!defined(_UNICOS)/0/' ${WRKSRC}/apps/speed.c > ${WRKSRC}/apps/speed.c.mod
+	mv ${WRKSRC}/apps/speed.c.mod ${WRKSRC}/apps/speed.c
+	
 .include "../../mk/bsd.pkg.mk"
