# HG changeset patch
# Parent d49f0b7d180b68c771d7fc4e6caa01f91b1be022

diff -r d49f0b7d180b security/libgpg-error/PLIST
--- a/security/libgpg-error/PLIST	Mon Feb 18 12:26:28 2013 -0500
+++ b/security/libgpg-error/PLIST	Mon Feb 18 12:26:33 2013 -0500
@@ -8,13 +8,3 @@
 share/common-lisp/source/gpg-error/gpg-error-package.lisp
 share/common-lisp/source/gpg-error/gpg-error.asd
 share/common-lisp/source/gpg-error/gpg-error.lisp
-${PLIST.nls}share/locale/cs/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/de/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/fr/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/it/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/nl/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/pl/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/ro/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/sv/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/vi/LC_MESSAGES/libgpg-error.mo
-${PLIST.nls}share/locale/zh_CN/LC_MESSAGES/libgpg-error.mo
