# HG changeset patch
# Parent 626a191f22c70f1192977a79a6bae2c53c6df023

diff -r 626a191f22c7 shells/bash/Makefile
--- a/shells/bash/Makefile	Mon Jul 08 19:22:17 2013 -0400
+++ b/shells/bash/Makefile	Mon Jul 08 19:22:22 2013 -0400
@@ -100,6 +100,9 @@
 CONFIGURE_ENV+=		bash_cv_wexitstatus_offset=8
 .endif
 
+# [Gregor] mischeck for musl, safe everywhere
+CONFIGURE_ARGS+=		--without-bash-malloc
+
 post-install:
 	${INSTALL_MAN} ${WRKSRC}/doc/bash.1 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1
 	${INSTALL_MAN} ${WRKSRC}/doc/bashbug.1 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1
