# HG changeset patch
# Parent 94f8d22d6ee8c21aacbefc7efd47b5b960143053

diff -r 94f8d22d6ee8 shells/bash/Makefile
--- a/shells/bash/Makefile	Sat Mar 14 00:04:41 2015 -0400
+++ b/shells/bash/Makefile	Sat Mar 14 00:04:47 2015 -0400
@@ -104,6 +104,9 @@
 pre-configure:
 	${RM} -f ${WRKSRC}/y.tab.c
 
+# [Gregor] mischeck for musl, safe everywhere
+CONFIGURE_ARGS+=		--without-bash-malloc
+
 post-install:
 	${INSTALL_MAN} ${WRKSRC}/doc/bash.1 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1
 	${INSTALL_MAN} ${WRKSRC}/doc/bashbug.1 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1
