# HG changeset patch
# Parent 3e2bf37f9167a654045c22d1ad038ce3aa766ed3

diff -r 3e2bf37f9167 sysutils/libpciaccess/distinfo
--- a/sysutils/libpciaccess/distinfo	Sat Mar 14 00:06:01 2015 -0400
+++ b/sysutils/libpciaccess/distinfo	Sat Mar 14 00:06:09 2015 -0400
@@ -6,3 +6,4 @@
 SHA1 (patch-configure) = 2b1bbc1ccd26372d7f45b0cbcf970823cafc6a7c
 SHA1 (patch-src_common__interface.c) = cac11689864531fa4721e09374ead730bd1691a6
 SHA1 (patch-src_solx__devfs.c) = 4800fde06d511592bbc4448f09656ee4d7abce2c
+SHA1 (patch-qq) = f91b1ee7151c127f3f087c2dbe3a7c6e4cbdc0e4
diff -r 3e2bf37f9167 sysutils/libpciaccess/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/sysutils/libpciaccess/patches/patch-qq	Sat Mar 14 00:06:09 2015 -0400
@@ -0,0 +1,23 @@
+diff -r 4ecbc201fcdd src/linux_sysfs.c
+--- src/linux_sysfs.c	Mon Mar 11 04:36:42 2013 +0000
++++ src/linux_sysfs.c	Mon Mar 11 04:40:52 2013 +0000
+@@ -44,6 +44,7 @@
+ #include <sys/mman.h>
+ #include <dirent.h>
+ #include <errno.h>
++#include <limits.h>
+ 
+ #if defined(__i386__) || defined(__x86_64__) || defined(__arm__)
+ #include <sys/io.h>
+diff -r 4ecbc201fcdd src/pciaccess_private.h
+--- src/pciaccess_private.h	Mon Mar 11 04:36:42 2013 +0000
++++ src/pciaccess_private.h	Mon Mar 11 04:40:52 2013 +0000
+@@ -42,7 +42,7 @@
+  * know of any OS we support where this isn't available in a sufficiently
+  * new version, so warn unconditionally.
+  */
+-#include <sys/fcntl.h>
++#include <fcntl.h>
+ 
+ #ifndef O_CLOEXEC
+ #warning O_CLOEXEC not available, please upgrade.
