# HG changeset patch
# Parent befc3c076d5147000fe0a0592f2e2982315ff882

diff -r befc3c076d51 sysutils/coreutils/Makefile
--- a/sysutils/coreutils/Makefile	Sat Mar 14 00:05:00 2015 -0400
+++ b/sysutils/coreutils/Makefile	Sat Mar 14 00:05:07 2015 -0400
@@ -65,6 +65,8 @@
 #MESSAGE_SUBST+=	OPSYS=${OPSYS:Q}
 #.endif
 
+CFLAGS+=		-DSLOW_BUT_NO_HACKS
+
 .include "../../converters/libiconv/buildlink3.mk"
 .include "../../devel/gettext-lib/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
diff -r befc3c076d51 sysutils/coreutils/distinfo
--- a/sysutils/coreutils/distinfo	Sat Mar 14 00:05:00 2015 -0400
+++ b/sysutils/coreutils/distinfo	Sat Mar 14 00:05:07 2015 -0400
@@ -7,3 +7,4 @@
 SHA1 (patch-lib-fflush.c) = 106e9c7d05fff290f24cb69e1b2c80d0077c62bd
 SHA1 (patch-lib-fseeko.c) = 9f4b0294698648011c64869fc0e642f910e9d47f
 SHA1 (patch-lib_string.in.h) = 37167af962d1cfe9196d11e4250d9e9628d9eb73
+SHA1 (patch-aq) = 02186df2372b3317cc2df6a8230cc198ac3eb4e9
diff -r befc3c076d51 sysutils/coreutils/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/sysutils/coreutils/patches/patch-aq	Sat Mar 14 00:05:07 2015 -0400
@@ -0,0 +1,23 @@
+diff -r fb0a1c120d18 lib/fseterr.c
+--- lib/fseterr.c	Sat May 05 16:00:33 2012 +0000
++++ lib/fseterr.c	Sat May 05 16:05:52 2012 +0000
+@@ -71,6 +71,6 @@
+     }
+   errno = saved_errno;
+ #else
+- #error "Please port gnulib fseterr.c to your platform! Look at the definitions of ferror and clearerr on your system, then report this to bug-gnulib."
++  return 1;
+ #endif
+ }
+diff -r fb0a1c120d18 src/dd.c
+--- src/dd.c	Sat May 05 16:00:33 2012 +0000
++++ src/dd.c	Sat May 05 16:05:52 2012 +0000
+@@ -1375,7 +1375,7 @@
+    The offending behavior has been confirmed with an Exabyte SCSI tape
+    drive accessed via /dev/nst0 on both Linux 2.2.17 and 2.4.16 kernels.  */
+ 
+-#ifdef __linux__
++#if 0
+ 
+ # include <sys/mtio.h>
+ 
