# HG changeset patch
# Parent 1e3376e02e19b77239d71c4a84e1c1e26802d103

diff -r 1e3376e02e19 sysutils/libpciaccess/distinfo
--- a/sysutils/libpciaccess/distinfo	Mon Apr 08 12:56:22 2013 -0400
+++ b/sysutils/libpciaccess/distinfo	Mon Apr 08 12:56:26 2013 -0400
@@ -5,3 +5,4 @@
 Size (libpciaccess-0.13.1.tar.bz2) = 352351 bytes
 SHA1 (patch-aa) = 090e2dd9d811fc56354d5387cc51fb2b1b4a2820
 SHA1 (patch-ac) = f860e9f7d99f381613f8f830281d157f2eb90905
+SHA1 (patch-qq) = f91b1ee7151c127f3f087c2dbe3a7c6e4cbdc0e4
diff -r 1e3376e02e19 sysutils/libpciaccess/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/sysutils/libpciaccess/patches/patch-qq	Mon Apr 08 12:56:26 2013 -0400
@@ -0,0 +1,23 @@
+diff -r 4ecbc201fcdd src/linux_sysfs.c
+--- src/linux_sysfs.c	Mon Mar 11 04:36:42 2013 +0000
++++ src/linux_sysfs.c	Mon Mar 11 04:40:52 2013 +0000
+@@ -44,6 +44,7 @@
+ #include <sys/mman.h>
+ #include <dirent.h>
+ #include <errno.h>
++#include <limits.h>
+ 
+ #if defined(__i386__) || defined(__x86_64__) || defined(__arm__)
+ #include <sys/io.h>
+diff -r 4ecbc201fcdd src/pciaccess_private.h
+--- src/pciaccess_private.h	Mon Mar 11 04:36:42 2013 +0000
++++ src/pciaccess_private.h	Mon Mar 11 04:40:52 2013 +0000
+@@ -42,7 +42,7 @@
+  * know of any OS we support where this isn't available in a sufficiently
+  * new version, so warn unconditionally.
+  */
+-#include <sys/fcntl.h>
++#include <fcntl.h>
+ 
+ #ifndef O_CLOEXEC
+ #warning O_CLOEXEC not available, please upgrade.
