# HG changeset patch
# Parent 003461f19a200e8ed32c70336091450a47f8e923

diff -r 003461f19a20 sysutils/dbus/distinfo
--- a/sysutils/dbus/distinfo	Mon Jul 08 19:25:37 2013 -0400
+++ b/sysutils/dbus/distinfo	Mon Jul 08 19:25:43 2013 -0400
@@ -9,3 +9,4 @@
 SHA1 (patch-al) = 3f9f793b0c1455bf2bf079cc27b841c22f42c276
 SHA1 (patch-am) = 8c794ff8b0981e90243ee20c26ae1ecc72e68de8
 SHA1 (patch-ba) = ceb8e0c424c57410f8c3775087e4cae1b44da4c5
+SHA1 (patch-qq) = 11a2511a66a12d886e444b7ca1745d66c97b1caa
diff -r 003461f19a20 sysutils/dbus/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/sysutils/dbus/patches/patch-qq	Mon Jul 08 19:25:43 2013 -0400
@@ -0,0 +1,12 @@
+diff -r 9389e1d2860b dbus/sd-daemon.c
+--- dbus/sd-daemon.c	Tue Nov 13 07:10:30 2012 +0000
++++ dbus/sd-daemon.c	Tue Nov 13 07:14:15 2012 +0000
+@@ -32,7 +32,7 @@
+ #include <sys/stat.h>
+ #include <sys/socket.h>
+ #include <sys/un.h>
+-#include <sys/fcntl.h>
++#include <fcntl.h>
+ #include <netinet/in.h>
+ #include <stdlib.h>
+ #include <errno.h>
