# HG changeset patch
# Parent f95baaf6fb4545f4e0b69f72dc849b78974e4b5b

diff -r f95baaf6fb45 textproc/libxml2/Makefile
--- a/textproc/libxml2/Makefile	Mon Apr 08 12:56:59 2013 -0400
+++ b/textproc/libxml2/Makefile	Mon Apr 08 12:57:04 2013 -0400
@@ -23,6 +23,9 @@
 
 PKGCONFIG_OVERRIDE=	libxml-2.0.pc.in
 
+# [Gregor] Threading support is broken on musl
+CONFIGURE_ARGS+=	--without-threads
+
 .include "options.mk"
 
 # allow thread-awareness, but make sure the library is not
