# HG changeset patch
# Parent 48de7d79616df177ef43618d2a0dcfdae96e8d74

diff -r 48de7d79616d textproc/libxml2/Makefile
--- a/textproc/libxml2/Makefile	Sat Mar 14 00:06:36 2015 -0400
+++ b/textproc/libxml2/Makefile	Sat Mar 14 00:06:42 2015 -0400
@@ -22,6 +22,9 @@
 
 PKGCONFIG_OVERRIDE=	libxml-2.0.pc.in
 
+# [Gregor] Threading support is broken on musl
+CONFIGURE_ARGS+=	--without-threads
+
 .include "options.mk"
 
 # allow thread-awareness, but make sure the library is not
