# HG changeset patch
# Parent 6dc177ad16457f13205721044208b945c0856d2d

diff -r 6dc177ad1645 textproc/libxml2/Makefile
--- a/textproc/libxml2/Makefile	Mon Feb 18 12:29:13 2013 -0500
+++ b/textproc/libxml2/Makefile	Mon Feb 18 12:29:18 2013 -0500
@@ -23,6 +23,9 @@
 
 PKGCONFIG_OVERRIDE=	libxml-2.0.pc.in
 
+# [Gregor] Threading support is broken on musl
+CONFIGURE_ARGS+=	--without-threads
+
 .include "options.mk"
 
 # allow thread-awareness, but make sure the library is not
