# HG changeset patch
# Parent 1ee1ca49dcca15415434785a9228e944579ce0a8

diff -r 1ee1ca49dcca www/firefox/Makefile
--- a/www/firefox/Makefile	Mon Apr 08 12:55:30 2013 -0400
+++ b/www/firefox/Makefile	Mon Apr 08 12:55:35 2013 -0400
@@ -20,7 +20,7 @@
 
 LDFLAGS.DragonFly=	-lplc4 -lnspr4
 LDFLAGS.FreeBSD=	-lplc4 -lnspr4
-LDFLAGS.Linux=		-lnspr4
+LDFLAGS.Linux=		-lplc4 -lnspr4
 
 .include "../../devel/xulrunner/mozilla-common.mk"
 .include "options.mk"
@@ -33,6 +33,9 @@
 MOZILLA_NAME=	Browser
 .endif
 
+# [Gregor] To make some patches work
+CFLAGS+=		-D__musl__
+
 pre-configure:
 	cd ${WRKSRC} && autoconf
 # XXX not built here, just to make check-portability.awk shut up.
