# HG changeset patch
# Parent 7e29e58c674206a4dcdba8610406db26bb37fb0f

diff -r 7e29e58c6742 www/libwww/Makefile
--- a/www/libwww/Makefile	Sat Mar 14 00:08:49 2015 -0400
+++ b/www/libwww/Makefile	Sat Mar 14 00:08:56 2015 -0400
@@ -24,8 +24,9 @@
 
 .include "options.mk"
 
+# [Gregor] Changed to zcat because busybox gets confused if you misname its things
 post-extract:
-	gzcat ${DISTDIR}/libwww-configure-5.4.0nb2.gz > ${WRKSRC}/configure
+	zcat ${DISTDIR}/libwww-configure-5.4.0nb2.gz > ${WRKSRC}/configure
 
 pre-configure:
 	chmod a+x ${WRKSRC}/configure
