# HG changeset patch
# Parent 8a2d65453a4685447ff0c7f5d0b74145708dbb48

diff -r 8a2d65453a46 www/webkit-gtk/Makefile
--- a/www/webkit-gtk/Makefile	Mon Apr 08 12:55:35 2013 -0400
+++ b/www/webkit-gtk/Makefile	Mon Apr 08 12:55:40 2013 -0400
@@ -10,5 +10,8 @@
 CONFIGURE_ARGS+=	--with-gtk=2.0
 CONFIGURE_ARGS+=	--disable-webkit2
 
+# [Gregor] For patches
+CFLAGS+=	-D__musl__
+
 .include "../../x11/gtk2/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
