# HG changeset patch
# Parent cd4f622a1ecf1841f8663644a88b1f3ed6c1e6a6

diff -r cd4f622a1ecf www/firefox/Makefile
--- a/www/firefox/Makefile	Tue Oct 09 17:56:39 2012 -0400
+++ b/www/firefox/Makefile	Tue Oct 09 17:56:45 2012 -0400
@@ -21,7 +21,7 @@
 
 LDFLAGS.DragonFly=	-lplc4 -lnspr4
 LDFLAGS.FreeBSD=	-lplc4 -lnspr4
-LDFLAGS.Linux=		-lnspr4
+LDFLAGS.Linux=		-lplc4 -lnspr4
 
 .include "../../devel/xulrunner/mozilla-common.mk"
 .include "options.mk"
@@ -34,6 +34,9 @@
 MOZILLA_NAME=	Browser
 .endif
 
+# [Gregor] To make some patches work
+CFLAGS+=		-D__musl__
+
 pre-configure:
 	cd ${WRKSRC} && autoconf
 # XXX not built here, just to make check-portability.awk shut up.
