# HG changeset patch
# Parent 8bd09242719209ed4d03fdcb230e6551bb12d26a

diff -r 8bd092427192 www/webkit-gtk/Makefile
--- a/www/webkit-gtk/Makefile	Mon Feb 18 12:27:42 2013 -0500
+++ b/www/webkit-gtk/Makefile	Mon Feb 18 12:27:46 2013 -0500
@@ -9,5 +9,8 @@
 CONFIGURE_ARGS+=	--with-gtk=2.0
 CONFIGURE_ARGS+=	--disable-webkit2
 
+# [Gregor] For patches
+CFLAGS+=	-D__musl__
+
 .include "../../x11/gtk2/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
