# HG changeset patch
# Parent ed7df63f9df1fe440c30e659932d852782da0b2f

diff -r ed7df63f9df1 www/firefox/Makefile
--- a/www/firefox/Makefile	Mon Feb 18 12:27:37 2013 -0500
+++ b/www/firefox/Makefile	Mon Feb 18 12:27:42 2013 -0500
@@ -20,7 +20,7 @@
 
 LDFLAGS.DragonFly=	-lplc4 -lnspr4
 LDFLAGS.FreeBSD=	-lplc4 -lnspr4
-LDFLAGS.Linux=		-lnspr4
+LDFLAGS.Linux=		-lplc4 -lnspr4
 
 .include "../../devel/xulrunner/mozilla-common.mk"
 .include "options.mk"
@@ -33,6 +33,9 @@
 MOZILLA_NAME=	Browser
 .endif
 
+# [Gregor] To make some patches work
+CFLAGS+=		-D__musl__
+
 pre-configure:
 	cd ${WRKSRC} && autoconf
 # XXX not built here, just to make check-portability.awk shut up.
