# HG changeset patch
# Parent 697e6ef32bd22810918528bfb0e87f4ef8281553

diff -r 697e6ef32bd2 x11/sessreg/distinfo
--- a/x11/sessreg/distinfo	Tue Oct 09 17:58:46 2012 -0400
+++ b/x11/sessreg/distinfo	Tue Oct 09 17:58:52 2012 -0400
@@ -5,3 +5,4 @@
 Size (sessreg-1.0.7.tar.bz2) = 120461 bytes
 SHA1 (patch-ab) = f4f8a51bfe022f6e4f8ad2471f13d68570f456cb
 SHA1 (patch-sessreg.c) = 1c05c8a4967527cd4ef8219cfd99eb3f7cce2489
+SHA1 (patch-zq) = bf1052f11c38daaeb281d5bd3166e40f46dcfc14
diff -r 697e6ef32bd2 x11/sessreg/patches/patch-zq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/sessreg/patches/patch-zq	Tue Oct 09 17:58:52 2012 -0400
@@ -0,0 +1,25 @@
+diff -r 4fdc7ea325ec sessreg.c
+--- sessreg.c	Mon Aug 06 21:31:43 2012 +0000
++++ sessreg.c	Mon Aug 06 21:33:44 2012 +0000
+@@ -99,10 +99,10 @@
+ static const char *utmpx_file = NULL;
+ #endif
+ #endif
+-#ifndef WTMPX_FILE
++#if !defined(WTMPX_FILE) && defined(_PATH_WTMPX)
+ #define WTMPX_FILE	_PATH_WTMPX
+ #endif
+-#ifndef UTMPX_FILE
++#if !defined(UTMPX_FILE) && defined(_PATH_UTMPX)
+ #define UTMPX_FILE	_PATH_UTMPX
+ #endif
+ 
+@@ -260,7 +260,7 @@
+ 	/* set up default file names */
+ 	if (!wflag) {
+ 		wtmp_file = WTMP_FILE;
+-#if defined(USE_UTMPX) && defined(HAVE_UPDWTMPX)
++#if defined(USE_UTMPX) && defined(HAVE_UPDWTMPX) && defined(WTMPX_FILE)
+ 		wtmpx_file = WTMPX_FILE;
+ #endif
+ 	}
