# HG changeset patch
# Parent 984912b3b9cfece59551178de609664627d0e755

diff -r 984912b3b9cf x11/openmotif/Makefile
--- a/x11/openmotif/Makefile	Sat Mar 14 00:07:29 2015 -0400
+++ b/x11/openmotif/Makefile	Sat Mar 14 00:07:36 2015 -0400
@@ -35,6 +35,9 @@
 
 CONFLICTS+=		lesstif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config
 GNU_CONFIGURE=		yes
