# HG changeset patch
# Parent 0c6d56c0ed04e490b4267af98e4b0ad3ae73a9ce

diff -r 0c6d56c0ed04 x11/modular-xorg-server/Makefile
--- a/x11/modular-xorg-server/Makefile	Mon Jul 08 19:24:03 2013 -0400
+++ b/x11/modular-xorg-server/Makefile	Mon Jul 08 19:24:09 2013 -0400
@@ -128,6 +128,9 @@
 BUILD_DEPENDS+=		tradcpp-[0-9]*:../../devel/tradcpp
 CONFIGURE_ENV+=		ac_cv_path_RAWCPP="${PREFIX}/bin/tradcpp -Uunix"
 
+# [Gregor] X-ACE doesn't work for unknown reasons
+CONFIGURE_ARGS+=	--disable-xace
+
 .include "options.mk"
 
 .include "../../devel/ncurses/buildlink3.mk"
diff -r 0c6d56c0ed04 x11/modular-xorg-server/distinfo
--- a/x11/modular-xorg-server/distinfo	Mon Jul 08 19:24:03 2013 -0400
+++ b/x11/modular-xorg-server/distinfo	Mon Jul 08 19:24:09 2013 -0400
@@ -6,3 +6,4 @@
 SHA1 (patch-configure) = 031bc0accf1dd71ed687e7aac3fcc9498cb06784
 SHA1 (patch-hw_xfree86_common_compiler.h) = 88de747715b80b22f448869ab32dce1641bcb6db
 SHA1 (patch-hw_xfree86_os-support_xf86__OSlib.h) = 7c2760509610b44915da077b22899bef9c338cc1
+SHA1 (patch-qq) = 671f0f246a4570a73c5ec0aa7008d9dea3410108
diff -r 0c6d56c0ed04 x11/modular-xorg-server/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/modular-xorg-server/patches/patch-qq	Mon Jul 08 19:24:09 2013 -0400
@@ -0,0 +1,18 @@
+diff -r a2b9db9818a9 hw/xfree86/os-support/xf86_OSlib.h
+--- hw/xfree86/os-support/xf86_OSlib.h	Mon Mar 11 15:21:01 2013 +0000
++++ hw/xfree86/os-support/xf86_OSlib.h	Mon Mar 11 16:36:31 2013 +0000
+@@ -215,11 +215,14 @@
+ # include <sys/types.h>
+ # include <assert.h>
+ 
++#if 0
+ # ifdef __linux__
+ #  include <termio.h>
+ # else /* __GLIBC__ */
+ #  include <termios.h>
+ # endif
++#endif
++#include <termios.h>
+ # ifdef __sparc__
+ #  include <sys/param.h>
+ # endif
