# HG changeset patch
# Parent 1ec0b7cad87f854b5dfd70088c3b2e8b6c1dc7a6

diff -r 1ec0b7cad87f x11/pixman/Makefile
--- a/x11/pixman/Makefile	Mon Feb 18 12:28:13 2013 -0500
+++ b/x11/pixman/Makefile	Mon Feb 18 12:28:18 2013 -0500
@@ -17,5 +17,8 @@
 
 PKGCONFIG_OVERRIDE+=	pixman-1.pc.in
 PKGCONFIG_OVERRIDE+=	pixman-1-uninstalled.pc.in
+	
+pre-configure:
+	sed -i 's/support_for__thread=yes/support_for__thread=no/' ${WRKSRC}/configure	
 
 .include "../../mk/bsd.pkg.mk"
