# HG changeset patch
# Parent 31adc158d03b916691cf9d2b0be04cfb30ef5a52

diff -r 31adc158d03b x11/sessreg/distinfo
--- a/x11/sessreg/distinfo	Sat Mar 14 00:06:49 2015 -0400
+++ b/x11/sessreg/distinfo	Sat Mar 14 00:06:56 2015 -0400
@@ -3,3 +3,4 @@
 SHA1 (sessreg-1.0.8.tar.bz2) = dc4fa8b804e7cb4cfbadbef3708c3b97076ddb65
 RMD160 (sessreg-1.0.8.tar.bz2) = f74a47dc735365093ce6152f0b035a7964830214
 Size (sessreg-1.0.8.tar.bz2) = 126979 bytes
+SHA1 (patch-zq) = bf1052f11c38daaeb281d5bd3166e40f46dcfc14
diff -r 31adc158d03b x11/sessreg/patches/patch-zq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/sessreg/patches/patch-zq	Sat Mar 14 00:06:56 2015 -0400
@@ -0,0 +1,25 @@
+diff -r 4fdc7ea325ec sessreg.c
+--- sessreg.c	Mon Aug 06 21:31:43 2012 +0000
++++ sessreg.c	Mon Aug 06 21:33:44 2012 +0000
+@@ -99,10 +99,10 @@
+ static const char *utmpx_file = NULL;
+ #endif
+ #endif
+-#ifndef WTMPX_FILE
++#if !defined(WTMPX_FILE) && defined(_PATH_WTMPX)
+ #define WTMPX_FILE	_PATH_WTMPX
+ #endif
+-#ifndef UTMPX_FILE
++#if !defined(UTMPX_FILE) && defined(_PATH_UTMPX)
+ #define UTMPX_FILE	_PATH_UTMPX
+ #endif
+ 
+@@ -260,7 +260,7 @@
+ 	/* set up default file names */
+ 	if (!wflag) {
+ 		wtmp_file = WTMP_FILE;
+-#if defined(USE_UTMPX) && defined(HAVE_UPDWTMPX)
++#if defined(USE_UTMPX) && defined(HAVE_UPDWTMPX) && defined(WTMPX_FILE)
+ 		wtmpx_file = WTMPX_FILE;
+ #endif
+ 	}
