# HG changeset patch
# Parent 5fc7410107cfb43e454e8b45b29704c52859cab7

diff -r 5fc7410107cf x11/qt4-libs/Makefile
--- a/x11/qt4-libs/Makefile	Sat Mar 14 00:07:51 2015 -0400
+++ b/x11/qt4-libs/Makefile	Sat Mar 14 00:07:58 2015 -0400
@@ -113,11 +113,12 @@
 		${WRKSRC}/plugins/bearer/libqgenericbearer.la \
 		${DESTDIR}${QTPREFIX}/plugins/bearer/
 #.endfor
-.for f in qcncodecs qjpcodecs qkrcodecs qtwcodecs
-	${LIBTOOL} --mode=install ${INSTALL_LIB} \
-		${WRKSRC}/plugins/codecs/lib${f}.la \
-		${DESTDIR}${QTPREFIX}/plugins/codecs/
-.endfor
+# [Gregor] We don't actually build these codecs on musl
+#.for f in qcncodecs qjpcodecs qkrcodecs qtwcodecs
+#	${LIBTOOL} --mode=install ${INSTALL_LIB} \
+#		${WRKSRC}/plugins/codecs/lib${f}.la \
+#		${DESTDIR}${QTPREFIX}/plugins/codecs/
+#.endfor
 .for f in qglgraphicssystem qtracegraphicssystem
 	${LIBTOOL} --mode=install ${INSTALL_LIB} \
 		${WRKSRC}/plugins/graphicssystems/lib${f}.la \
diff -r 5fc7410107cf x11/qt4-libs/PLIST
--- a/x11/qt4-libs/PLIST	Sat Mar 14 00:07:51 2015 -0400
+++ b/x11/qt4-libs/PLIST	Sat Mar 14 00:07:58 2015 -0400
@@ -46,10 +46,6 @@
 qt4/plugins/accessible/libqtaccessiblewidgets.la
 ${PLIST.mac}qt4/plugins/bearer/libqcorewlanbearer.la
 qt4/plugins/bearer/libqgenericbearer.la
-qt4/plugins/codecs/libqcncodecs.la
-qt4/plugins/codecs/libqjpcodecs.la
-qt4/plugins/codecs/libqkrcodecs.la
-qt4/plugins/codecs/libqtwcodecs.la
 qt4/plugins/graphicssystems/libqglgraphicssystem.la
 qt4/plugins/graphicssystems/libqtracegraphicssystem.la
 qt4/plugins/iconengines/libqsvgicon.la
diff -r 5fc7410107cf x11/qt4-libs/distinfo
--- a/x11/qt4-libs/distinfo	Sat Mar 14 00:07:51 2015 -0400
+++ b/x11/qt4-libs/distinfo	Sat Mar 14 00:07:58 2015 -0400
@@ -69,5 +69,6 @@
 SHA1 (patch-src_gui_kernel_qcocoaapplicationdelegate__mac.mm) = 0caa9b006b3ffee4ab747fca9fd224c7c49211c9
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols.cpp) = 3ad682b86d2e9bd2b282caa298508dc3e9dd8566
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols__p.h) = 417846ba9edab8638cafa41a54ef60029467ef80
+SHA1 (patch-qq) = 0b23302194af00685017cc4319c3b6cb10fc1fae
 SHA1 (patch-src_plugins_accessible_widgets_itemviews.cpp) = 1456fbaacef33f3b2422158d758d02990845048a
 SHA1 (patch-src_scripttools_debugging_qscriptdebuggerconsole.cpp) = eea948f07fd9e595c7813aa485e985407dfcfbf9
diff -r 5fc7410107cf x11/qt4-libs/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/qt4-libs/patches/patch-qq	Sat Mar 14 00:07:58 2015 -0400
@@ -0,0 +1,70 @@
+diff -r bf00ba1f1322 mkspecs/linux-g++/qplatformdefs.h
+--- mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 04:21:22 2013 +0000
++++ mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -86,11 +86,7 @@
+ 
+ #undef QT_SOCKLEN_T
+ 
+-#if defined(__GLIBC__) && (__GLIBC__ >= 2)
+ #define QT_SOCKLEN_T            socklen_t
+-#else
+-#define QT_SOCKLEN_T            int
+-#endif
+ 
+ #if defined(_XOPEN_SOURCE) && (_XOPEN_SOURCE >= 500)
+ #define QT_SNPRINTF		::snprintf
+diff -r bf00ba1f1322 src/3rdparty/clucene/src/CLucene/util/Misc.cpp
+--- src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 05:47:26 2013 +0000
+@@ -20,6 +20,7 @@
+ # endif
+ #endif
+ 
++#undef _CL_HAVE_SYS_TIMEB_H
+ #ifdef _CL_HAVE_SYS_TIMEB_H
+ # include <sys/timeb.h>
+ #endif
+diff -r bf00ba1f1322 src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -711,8 +711,10 @@
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !OS(AIX) && !OS(HPUX)
++#if !defined(__linux__) || defined(__GLIBC__)
+ #define HAVE_TM_GMTOFF 1
+ #define HAVE_TM_ZONE 1
++#endif
+ #define HAVE_TIMEGM 1
+ #endif
+ 
+diff -r bf00ba1f1322 src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -582,8 +582,10 @@
+ #endif
+ 
+ #if (PLATFORM(GTK) || PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(WIN) || (PLATFORM(QT) && (OS(DARWIN) || USE(PTHREAD_BASED_QT)) && !ENABLE(SINGLE_THREADED))) && !OS(QNX) && !defined(ENABLE_JSC_MULTIPLE_THREADS)
++#if 1
+ #define ENABLE_JSC_MULTIPLE_THREADS 1
+ #endif
++#endif
+ 
+ #if ENABLE(JSC_MULTIPLE_THREADS)
+ #define ENABLE_WTF_MULTIPLE_THREADS 1
+@@ -766,11 +768,13 @@
+ #endif
+ #endif
+ 
++#if defined(__GLIBC__)
++#define HAVE_TM_GMTOFF 1
++#define HAVE_TM_ZONE 1
++#endif
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !PLATFORM(BREWMP)
+-#define HAVE_TM_GMTOFF 1
+-#define HAVE_TM_ZONE 1
+ #define HAVE_TIMEGM 1
+ #endif
+ 
