diff -r 5a6b35833cab x11/modular-xorg-server/Makefile
--- a/x11/modular-xorg-server/Makefile	Sat Mar 14 00:06:09 2015 -0400
+++ b/x11/modular-xorg-server/Makefile	Sat Mar 14 00:06:16 2015 -0400
@@ -127,6 +127,9 @@
 BUILD_DEPENDS+=		tradcpp-[0-9]*:../../devel/tradcpp
 CONFIGURE_ENV+=		ac_cv_path_RAWCPP="${PREFIX}/bin/tradcpp -Uunix"
 
+# [Gregor] X-ACE doesn't work for unknown reasons
+CONFIGURE_ARGS+=	--disable-xace
+
 .include "options.mk"
 
 .include "../../devel/ncurses/buildlink3.mk"
diff -r 5a6b35833cab x11/modular-xorg-server/PLIST
--- a/x11/modular-xorg-server/PLIST	Sat Mar 14 00:06:09 2015 -0400
+++ b/x11/modular-xorg-server/PLIST	Sat Mar 14 00:06:16 2015 -0400
@@ -129,8 +129,6 @@
 include/xorg/xaarop.h
 include/xorg/xaa.h
 include/xorg/xaalocal.h
-include/xorg/xace.h
-include/xorg/xacestr.h
 include/xorg/xf86.h
 include/xorg/xf86Crtc.h
 include/xorg/xf86Cursor.h
diff -r 5a6b35833cab x11/modular-xorg-server/distinfo
--- a/x11/modular-xorg-server/distinfo	Sat Mar 14 00:06:09 2015 -0400
+++ b/x11/modular-xorg-server/distinfo	Sat Mar 14 00:06:16 2015 -0400
@@ -46,6 +46,7 @@
 SHA1 (patch-hw_xfree86_common_compiler.h) = 88de747715b80b22f448869ab32dce1641bcb6db
 SHA1 (patch-hw_xfree86_dri2_dri2ext.c) = bc2b57e4bef5ae386d1d3a015c01d243f8880ab4
 SHA1 (patch-hw_xfree86_os-support_xf86__OSlib.h) = 7c2760509610b44915da077b22899bef9c338cc1
+SHA1 (patch-qq) = 75364154810f7a805ded4e5da6f62ea7d91e05a8
 SHA1 (patch-include_dix.h) = ab0dc2debd87f29c0ac7f971a3cad4850b239c3a
 SHA1 (patch-include_regionstr.h) = c6f3e3b263593b622e7087d3ebe2b470cf9526e3
 SHA1 (patch-os_access.c) = 94ee087d68bc9ba713dbe42102f971a1b8e6a3c4
diff -r 5a6b35833cab x11/modular-xorg-server/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/modular-xorg-server/patches/patch-qq	Sat Mar 14 00:06:16 2015 -0400
@@ -0,0 +1,27 @@
+diff -r 9da799702a4b hw/xfree86/common/xf86Init.c
+--- hw/xfree86/common/xf86Init.c	Wed Jul 31 13:10:46 2013 +0000
++++ hw/xfree86/common/xf86Init.c	Wed Jul 31 13:42:05 2013 +0000
+@@ -29,6 +29,8 @@
+  * the sale, use or other dealings in this Software without prior written
+  * authorization from the copyright holder(s) and author(s).
+  */
++ 
++#define _GNU_SOURCE /* for getres[ug]id */
+ 
+ #ifdef HAVE_XORG_CONFIG_H
+ #include <xorg-config.h>
+diff -r 9da799702a4b hw/xfree86/os-support/xf86_OSlib.h
+--- hw/xfree86/os-support/xf86_OSlib.h	Wed Jul 31 13:10:46 2013 +0000
++++ hw/xfree86/os-support/xf86_OSlib.h	Wed Jul 31 13:42:05 2013 +0000
+@@ -178,11 +178,7 @@
+ #include <sys/types.h>
+ #include <assert.h>
+ 
+-#ifdef __linux__
+-#include <termio.h>
+-#else                           /* __GLIBC__ */
+ #include <termios.h>
+-#endif
+ #ifdef __sparc__
+ #include <sys/param.h>
+ #endif
