# HG changeset patch
# Parent 122f0f179b2a9183f9ca6a30e498d82a457919b4

diff -r 122f0f179b2a x11/pixman/Makefile
--- a/x11/pixman/Makefile	Mon Apr 08 12:56:06 2013 -0400
+++ b/x11/pixman/Makefile	Mon Apr 08 12:56:11 2013 -0400
@@ -16,5 +16,10 @@
 
 PKGCONFIG_OVERRIDE+=	pixman-1.pc.in
 PKGCONFIG_OVERRIDE+=	pixman-1-uninstalled.pc.in
+	
+pre-configure:
+	sed 's/support_for__thread=yes/support_for__thread=no/' ${WRKSRC}/configure > ${WRKSRC}/configure.mod
+	mv ${WRKSRC}/configure.mod ${WRKSRC}/configure
+	chmod a+x ${WRKSRC}/configure
 
 .include "../../mk/bsd.pkg.mk"
