# HG changeset patch
# Parent cb41b8ed748caf7b38a57d5ce9c2222c0df194a4

diff -r cb41b8ed748c x11/libXfont/Makefile
--- a/x11/libXfont/Makefile	Mon Jul 08 19:23:18 2013 -0400
+++ b/x11/libXfont/Makefile	Mon Jul 08 19:23:23 2013 -0400
@@ -19,6 +19,7 @@
 PKGCONFIG_OVERRIDE+=	xfont.pc.in
 
 CONFIGURE_ARGS+=	--disable-devel-docs
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -DNOFILES_MAX=256"
 
 .include "../../meta-pkgs/modular-xorg/hacks.mk"
 
