# HG changeset patch
# Parent c291b6f2484c1f04dc9bf7cd1d32235e46786a7e

diff -r c291b6f2484c x11/xload/distinfo
--- a/x11/xload/distinfo	Sat Mar 14 00:06:56 2015 -0400
+++ b/x11/xload/distinfo	Sat Mar 14 00:07:02 2015 -0400
@@ -3,3 +3,4 @@
 SHA1 (xload-1.1.2.tar.bz2) = 4a34135ff51c3bc4029d6644e60e03f17ccf8c81
 RMD160 (xload-1.1.2.tar.bz2) = 32d8999cee6c9fe5b3f1a5be7c0d15d875474ff0
 Size (xload-1.1.2.tar.bz2) = 136745 bytes
+SHA1 (patch-aq) = 91846822d3403048ca2b9b9ace3314bd1b880ebe
diff -r c291b6f2484c x11/xload/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/xload/patches/patch-aq	Sat Mar 14 00:07:02 2015 -0400
@@ -0,0 +1,25 @@
+diff -r 51a31ec23212 get_rload.c
+--- get_rload.c	Sat Aug 11 06:34:00 2012 +0000
++++ get_rload.c	Sat Aug 11 06:36:34 2012 +0000
+@@ -9,7 +9,7 @@
+ /* Not all OS supports get_rload
+    steal the STUB idea from get_load
+  */
+-#if defined(QNX4) || defined(__CYGWIN__)
++#if defined(QNX4) || defined(__CYGWIN__) || 1
+ #define RLOADSTUB
+ #endif
+ 
+diff -r 51a31ec23212 xload.c
+--- xload.c	Sat Aug 11 06:34:00 2012 +0000
++++ xload.c	Sat Aug 11 06:36:34 2012 +0000
+@@ -77,7 +77,9 @@
+ 
+ #ifdef USE_GETTEXT
+ # include <X11/Xlocale.h>
++#if 0
+ # include <libintl.h>
++#endif
+ #else
+ # define gettext(a) (a)
+ #endif
