# HG changeset patch
# Parent 104f99e147e96f66fd45629dae3571e3ea43aefa

diff -r 104f99e147e9 x11/libdrm/distinfo
--- a/x11/libdrm/distinfo	Tue Oct 09 17:57:03 2012 -0400
+++ b/x11/libdrm/distinfo	Tue Oct 09 17:57:08 2012 -0400
@@ -9,3 +9,4 @@
 SHA1 (patch-include_drm_drm.h) = d370e83da5c39f0e49094787c5f4580758c911b0
 SHA1 (patch-intel_intel__bufmgr__gem.c) = 418d00f24344b7abaa86d3b3d10817314ce70175
 SHA1 (patch-libkms_linux.c) = 737d0a8e7b742ee9d24fec94c72de6b4915dd675
+SHA1 (patch-aq) = 87e114775f5761986528f8c0c87ebaefe52d593b
diff -r 104f99e147e9 x11/libdrm/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/libdrm/patches/patch-aq	Tue Oct 09 17:57:08 2012 -0400
@@ -0,0 +1,12 @@
+diff -r 3f68fbb9449a intel/intel_bufmgr_fake.c
+--- intel/intel_bufmgr_fake.c	Mon May 07 20:01:11 2012 +0000
++++ intel/intel_bufmgr_fake.c	Mon May 07 20:01:34 2012 +0000
+@@ -51,6 +51,8 @@
+ #include "mm.h"
+ #include "libdrm_lists.h"
+ 
++#include <strings.h>
++
+ /* Support gcc's __FUNCTION__ for people using other compilers */
+ #if !defined(__GNUC__) && !defined(__FUNCTION__)
+ # define __FUNCTION__ __func__ /* C99 */
