# HG changeset patch
# Parent 13466d561bb20437694079e59794a77ec36aa2d1

diff -r 13466d561bb2 x11/qt4-libs/Makefile
--- a/x11/qt4-libs/Makefile	Mon Jul 08 19:26:30 2013 -0400
+++ b/x11/qt4-libs/Makefile	Mon Jul 08 19:26:36 2013 -0400
@@ -99,11 +99,12 @@
 	${LIBTOOL} --mode=install ${INSTALL_LIB} \
 		${WRKSRC}/plugins/bearer/libqgenericbearer.la \
 		${DESTDIR}${QTPREFIX}/plugins/bearer/
-.for codec in qcncodecs qjpcodecs qkrcodecs qtwcodecs
-	${LIBTOOL} --mode=install ${INSTALL_LIB} \
-		${WRKSRC}/plugins/codecs/lib${codec}.la \
-		${DESTDIR}${QTPREFIX}/plugins/codecs/
-.endfor
+# [Gregor] We don't actually build these codecs on musl
+#.for codec in qcncodecs qjpcodecs qkrcodecs qtwcodecs
+#	${LIBTOOL} --mode=install ${INSTALL_LIB} \
+#		${WRKSRC}/plugins/codecs/lib${codec}.la \
+#		${DESTDIR}${QTPREFIX}/plugins/codecs/
+#.endfor
 .for plug in qglgraphicssystem qtracegraphicssystem
 	${LIBTOOL} --mode=install ${INSTALL_LIB} \
 		${WRKSRC}/plugins/graphicssystems/lib${plug}.la \
diff -r 13466d561bb2 x11/qt4-libs/PLIST
--- a/x11/qt4-libs/PLIST	Mon Jul 08 19:26:30 2013 -0400
+++ b/x11/qt4-libs/PLIST	Mon Jul 08 19:26:36 2013 -0400
@@ -44,10 +44,6 @@
 qt4/plugins/accessible/libqtaccessiblecompatwidgets.la
 qt4/plugins/accessible/libqtaccessiblewidgets.la
 qt4/plugins/bearer/libqgenericbearer.la
-qt4/plugins/codecs/libqcncodecs.la
-qt4/plugins/codecs/libqjpcodecs.la
-qt4/plugins/codecs/libqkrcodecs.la
-qt4/plugins/codecs/libqtwcodecs.la
 qt4/plugins/graphicssystems/libqglgraphicssystem.la
 qt4/plugins/graphicssystems/libqtracegraphicssystem.la
 qt4/plugins/iconengines/libqsvgicon.la
diff -r 13466d561bb2 x11/qt4-libs/distinfo
--- a/x11/qt4-libs/distinfo	Mon Jul 08 19:26:30 2013 -0400
+++ b/x11/qt4-libs/distinfo	Mon Jul 08 19:26:36 2013 -0400
@@ -78,3 +78,4 @@
 SHA1 (patch-src_gui_kernel_qt_cocoa_helpers_mac_p.h) = 84d356628d764797524fe72139ea6829a33524ea
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols.cpp) = 3ad682b86d2e9bd2b282caa298508dc3e9dd8566
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols__p.h) = 417846ba9edab8638cafa41a54ef60029467ef80
+SHA1 (patch-qq) = 0b23302194af00685017cc4319c3b6cb10fc1fae
diff -r 13466d561bb2 x11/qt4-libs/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/qt4-libs/patches/patch-qq	Mon Jul 08 19:26:36 2013 -0400
@@ -0,0 +1,70 @@
+diff -r bf00ba1f1322 mkspecs/linux-g++/qplatformdefs.h
+--- mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 04:21:22 2013 +0000
++++ mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -86,11 +86,7 @@
+ 
+ #undef QT_SOCKLEN_T
+ 
+-#if defined(__GLIBC__) && (__GLIBC__ >= 2)
+ #define QT_SOCKLEN_T            socklen_t
+-#else
+-#define QT_SOCKLEN_T            int
+-#endif
+ 
+ #if defined(_XOPEN_SOURCE) && (_XOPEN_SOURCE >= 500)
+ #define QT_SNPRINTF		::snprintf
+diff -r bf00ba1f1322 src/3rdparty/clucene/src/CLucene/util/Misc.cpp
+--- src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 05:47:26 2013 +0000
+@@ -20,6 +20,7 @@
+ # endif
+ #endif
+ 
++#undef _CL_HAVE_SYS_TIMEB_H
+ #ifdef _CL_HAVE_SYS_TIMEB_H
+ # include <sys/timeb.h>
+ #endif
+diff -r bf00ba1f1322 src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -711,8 +711,10 @@
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !OS(AIX) && !OS(HPUX)
++#if !defined(__linux__) || defined(__GLIBC__)
+ #define HAVE_TM_GMTOFF 1
+ #define HAVE_TM_ZONE 1
++#endif
+ #define HAVE_TIMEGM 1
+ #endif
+ 
+diff -r bf00ba1f1322 src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -582,8 +582,10 @@
+ #endif
+ 
+ #if (PLATFORM(GTK) || PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(WIN) || (PLATFORM(QT) && (OS(DARWIN) || USE(PTHREAD_BASED_QT)) && !ENABLE(SINGLE_THREADED))) && !OS(QNX) && !defined(ENABLE_JSC_MULTIPLE_THREADS)
++#if 1
+ #define ENABLE_JSC_MULTIPLE_THREADS 1
+ #endif
++#endif
+ 
+ #if ENABLE(JSC_MULTIPLE_THREADS)
+ #define ENABLE_WTF_MULTIPLE_THREADS 1
+@@ -766,11 +768,13 @@
+ #endif
+ #endif
+ 
++#if defined(__GLIBC__)
++#define HAVE_TM_GMTOFF 1
++#define HAVE_TM_ZONE 1
++#endif
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !PLATFORM(BREWMP)
+-#define HAVE_TM_GMTOFF 1
+-#define HAVE_TM_ZONE 1
+ #define HAVE_TIMEGM 1
+ #endif
+ 
