# HG changeset patch
# Parent fd0b52bcd2fc781329f99394572756a151563c51

diff -r fd0b52bcd2fc x11/pixman/Makefile
--- a/x11/pixman/Makefile	Mon Jul 08 19:23:40 2013 -0400
+++ b/x11/pixman/Makefile	Mon Jul 08 19:23:46 2013 -0400
@@ -21,5 +21,10 @@
 
 PKGCONFIG_OVERRIDE+=	pixman-1.pc.in
 PKGCONFIG_OVERRIDE+=	pixman-1-uninstalled.pc.in
+	
+pre-configure:
+	sed 's/support_for__thread=yes/support_for__thread=no/' ${WRKSRC}/configure > ${WRKSRC}/configure.mod
+	mv ${WRKSRC}/configure.mod ${WRKSRC}/configure
+	chmod a+x ${WRKSRC}/configure
 
 .include "../../mk/bsd.pkg.mk"
