# HG changeset patch
# Parent cb99470592b1360ba3dc69270d577e81fbb767be

diff -r cb99470592b1 x11/motif/Makefile
--- a/x11/motif/Makefile	Sat Mar 14 00:08:10 2015 -0400
+++ b/x11/motif/Makefile	Sat Mar 14 00:08:17 2015 -0400
@@ -19,6 +19,9 @@
 
 CONFLICTS+=		lesstif-[0-9]* openmotif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config automake yacc
 GNU_CONFIGURE=		yes
