# HG changeset patch
# Parent b09b2f3502ce09a459132706b3bf15d957090e80

diff -r b09b2f3502ce x11/openmotif/Makefile
--- a/x11/openmotif/Makefile	Mon Jul 08 19:26:00 2013 -0400
+++ b/x11/openmotif/Makefile	Mon Jul 08 19:26:06 2013 -0400
@@ -37,6 +37,9 @@
 
 CONFLICTS+=		lesstif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config
 GNU_CONFIGURE=		yes
