# HG changeset patch
# Parent f8e0d97ac52bb9cc341d0284e3b192904247377c

diff -r f8e0d97ac52b x11/openmotif/Makefile
--- a/x11/openmotif/Makefile	Mon Feb 18 12:30:44 2013 -0500
+++ b/x11/openmotif/Makefile	Mon Feb 18 12:30:49 2013 -0500
@@ -37,6 +37,9 @@
 
 CONFLICTS+=		lesstif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config
 GNU_CONFIGURE=		yes
