# HG changeset patch
# Parent 20c0cae032276b3d984445b13752b89af7ba161d

diff -r 20c0cae03227 x11/openmotif/Makefile
--- a/x11/openmotif/Makefile	Mon Apr 08 12:58:13 2013 -0400
+++ b/x11/openmotif/Makefile	Mon Apr 08 12:58:18 2013 -0400
@@ -37,6 +37,9 @@
 
 CONFLICTS+=		lesstif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config
 GNU_CONFIGURE=		yes
