# HG changeset patch
# Parent 103ad873ef18486110e55c1c65a24e5722b8b4af

diff -r 103ad873ef18 x11/fltk/distinfo
--- a/x11/fltk/distinfo	Mon Jul 08 19:25:31 2013 -0400
+++ b/x11/fltk/distinfo	Mon Jul 08 19:25:37 2013 -0400
@@ -11,3 +11,4 @@
 SHA1 (patch-af) = 791ba33cfc85e41e0f2b75ce0d1b0d6a378d9d0b
 SHA1 (patch-ag) = aaeb376acf79b695e8376e7610534d125dff6a86
 SHA1 (patch-ah) = 876f79b4d0523035307ecc7d76460cad44b40704
+SHA1 (patch-aq) = 4d24b7faa438c07712456849145c9a868909c1b9
diff -r 103ad873ef18 x11/fltk/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/fltk/patches/patch-aq	Mon Jul 08 19:25:37 2013 -0400
@@ -0,0 +1,12 @@
+diff -r f188cac510dd src/filename_list.cxx
+--- src/filename_list.cxx	Sun Aug 26 13:50:57 2012 +0000
++++ src/filename_list.cxx	Sun Aug 26 13:54:01 2012 +0000
+@@ -67,7 +67,7 @@
+   // The vast majority of UNIX systems want the sort function to have this
+   // prototype, most likely so that it can be passed to qsort without any
+   // changes:
+-  int n = scandir(d, list, 0, (int(*)(const void*,const void*))sort);
++  int n = scandir(d, list, 0, (int(*)(const dirent **,const dirent **))sort);
+ #else
+   // This version is when we define our own scandir (WIN32 and perhaps
+   // some Unix systems) and apparently on IRIX:
