# HG changeset patch
# Parent f18df778bf64850493668bdc8141ba59098feaa3
diff -r f18df778bf64 -r 8bbef41e5d43 x11/qt4-libs/Makefile
--- a/x11/qt4-libs/Makefile	Sat Mar 09 07:10:20 2013 -0500
+++ b/x11/qt4-libs/Makefile	Sat Mar 09 20:28:50 2013 -0500
@@ -93,11 +93,12 @@
 	${LIBTOOL} --mode=install ${INSTALL_LIB} \
 		${WRKSRC}/plugins/bearer/libqgenericbearer.la \
 		${DESTDIR}${QTPREFIX}/plugins/bearer/
-.for codec in qcncodecs qjpcodecs qkrcodecs qtwcodecs
-	${LIBTOOL} --mode=install ${INSTALL_LIB} \
-		${WRKSRC}/plugins/codecs/lib${codec}.la \
-		${DESTDIR}${QTPREFIX}/plugins/codecs/
-.endfor
+# [Gregor] We don't actually build these codecs on musl
+#.for codec in qcncodecs qjpcodecs qkrcodecs qtwcodecs
+#	${LIBTOOL} --mode=install ${INSTALL_LIB} \
+#		${WRKSRC}/plugins/codecs/lib${codec}.la \
+#		${DESTDIR}${QTPREFIX}/plugins/codecs/
+#.endfor
 .for plug in qglgraphicssystem qtracegraphicssystem
 	${LIBTOOL} --mode=install ${INSTALL_LIB} \
 		${WRKSRC}/plugins/graphicssystems/lib${plug}.la \
diff -r f18df778bf64 -r 8bbef41e5d43 x11/qt4-libs/PLIST
--- a/x11/qt4-libs/PLIST	Sat Mar 09 07:10:20 2013 -0500
+++ b/x11/qt4-libs/PLIST	Sat Mar 09 20:28:50 2013 -0500
@@ -44,10 +44,6 @@
 qt4/plugins/accessible/libqtaccessiblecompatwidgets.la
 qt4/plugins/accessible/libqtaccessiblewidgets.la
 qt4/plugins/bearer/libqgenericbearer.la
-qt4/plugins/codecs/libqcncodecs.la
-qt4/plugins/codecs/libqjpcodecs.la
-qt4/plugins/codecs/libqkrcodecs.la
-qt4/plugins/codecs/libqtwcodecs.la
 qt4/plugins/graphicssystems/libqglgraphicssystem.la
 qt4/plugins/graphicssystems/libqtracegraphicssystem.la
 qt4/plugins/iconengines/libqsvgicon.la
diff -r f18df778bf64 -r 8bbef41e5d43 x11/qt4-libs/distinfo
--- a/x11/qt4-libs/distinfo	Sat Mar 09 07:10:20 2013 -0500
+++ b/x11/qt4-libs/distinfo	Sat Mar 09 20:28:50 2013 -0500
@@ -55,3 +55,4 @@
 SHA1 (patch-src_corelib_io_qfilesystemwatcher.cpp) = bb16b95d20286b1aa069dc25843d7e0067cc0268
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols.cpp) = 3ad682b86d2e9bd2b282caa298508dc3e9dd8566
 SHA1 (patch-src_network_ssl_qsslsocket__openssl__symbols__p.h) = 417846ba9edab8638cafa41a54ef60029467ef80
+SHA1 (patch-qq) = 13a9a50ab4a8e0786315b3c45ac098c145611160
diff -r f18df778bf64 -r 8bbef41e5d43 x11/qt4-libs/patches/patch-qq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/qt4-libs/patches/patch-qq	Sat Mar 09 20:28:50 2013 -0500
@@ -0,0 +1,137 @@
+diff -r bf00ba1f1322 mkspecs/linux-g++/qplatformdefs.h
+--- mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 04:21:22 2013 +0000
++++ mkspecs/linux-g++/qplatformdefs.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -86,11 +86,7 @@
+ 
+ #undef QT_SOCKLEN_T
+ 
+-#if defined(__GLIBC__) && (__GLIBC__ >= 2)
+ #define QT_SOCKLEN_T            socklen_t
+-#else
+-#define QT_SOCKLEN_T            int
+-#endif
+ 
+ #if defined(_XOPEN_SOURCE) && (_XOPEN_SOURCE >= 500)
+ #define QT_SNPRINTF		::snprintf
+diff -r bf00ba1f1322 src/3rdparty/clucene/src/CLucene/util/Misc.cpp
+--- src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/clucene/src/CLucene/util/Misc.cpp	Wed Mar 06 05:47:26 2013 +0000
+@@ -20,6 +20,7 @@
+ # endif
+ #endif
+ 
++#undef _CL_HAVE_SYS_TIMEB_H
+ #ifdef _CL_HAVE_SYS_TIMEB_H
+ # include <sys/timeb.h>
+ #endif
+diff -r bf00ba1f1322 src/3rdparty/javascriptcore/JavaScriptCore/runtime/Collector.cpp
+--- src/3rdparty/javascriptcore/JavaScriptCore/runtime/Collector.cpp	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/javascriptcore/JavaScriptCore/runtime/Collector.cpp	Wed Mar 06 05:47:26 2013 +0000
+@@ -657,6 +657,25 @@
+ #if HAVE(PTHREAD_NP_H) || OS(NETBSD)
+         // e.g. on FreeBSD 5.4, neundorf@kde.org
+         pthread_attr_get_np(thread, &sattr);
++#elif defined(__linux__) && !defined(__GLIBC__)
++        {
++            unsigned long inStack;
++            FILE *maps;
++            
++            inStack = (unsigned long) &inStack;
++            maps = fopen("/proc/self/maps", "r");
++            if (maps != NULL) {
++                unsigned long start, end;
++                while (fscanf(maps, "%lx-%lx%*[^\n]\n", &start, &end) >= 2) {
++                    if (start <= inStack && end > inStack) {
++                        // found it
++                        stackBase = (void *) end; // FIXME: assuming we grow down
++                        break;
++                    }
++                }
++                fclose(maps);
++            }
++        }
+ #else
+         // FIXME: this function is non-portable; other POSIX systems may have different np alternatives
+         pthread_getattr_np(thread, &sattr);
+diff -r bf00ba1f1322 src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -711,8 +711,10 @@
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !OS(AIX) && !OS(HPUX)
++#if !defined(__linux__) || defined(__GLIBC__)
+ #define HAVE_TM_GMTOFF 1
+ #define HAVE_TM_ZONE 1
++#endif
+ #define HAVE_TIMEGM 1
+ #endif
+ 
+diff -r bf00ba1f1322 src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h
+--- src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	Wed Mar 06 05:47:26 2013 +0000
+@@ -582,8 +582,10 @@
+ #endif
+ 
+ #if (PLATFORM(GTK) || PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(WIN) || (PLATFORM(QT) && (OS(DARWIN) || USE(PTHREAD_BASED_QT)) && !ENABLE(SINGLE_THREADED))) && !OS(QNX) && !defined(ENABLE_JSC_MULTIPLE_THREADS)
++#if !defined(__linux__) || defined(__GLIBC__)
+ #define ENABLE_JSC_MULTIPLE_THREADS 1
+ #endif
++#endif
+ 
+ #if ENABLE(JSC_MULTIPLE_THREADS)
+ #define ENABLE_WTF_MULTIPLE_THREADS 1
+@@ -766,11 +768,13 @@
+ #endif
+ #endif
+ 
++#if defined(__GLIBC__)
++#define HAVE_TM_GMTOFF 1
++#define HAVE_TM_ZONE 1
++#endif
+ #if !OS(WINDOWS) && !OS(SOLARIS) && !OS(QNX) \
+     && !OS(SYMBIAN) && !OS(HAIKU) && !OS(RVCT) \
+     && !OS(ANDROID) && !PLATFORM(BREWMP)
+-#define HAVE_TM_GMTOFF 1
+-#define HAVE_TM_ZONE 1
+ #define HAVE_TIMEGM 1
+ #endif
+ 
+diff -r bf00ba1f1322 src/3rdparty/webkit/Source/JavaScriptCore/wtf/StackBounds.cpp
+--- src/3rdparty/webkit/Source/JavaScriptCore/wtf/StackBounds.cpp	Wed Mar 06 04:21:22 2013 +0000
++++ src/3rdparty/webkit/Source/JavaScriptCore/wtf/StackBounds.cpp	Wed Mar 06 05:47:26 2013 +0000
+@@ -21,6 +21,8 @@
+ #include "config.h"
+ #include "StackBounds.h"
+ 
++#include <cstdio>
++
+ #if OS(DARWIN)
+ 
+ #include <mach/task.h>
+@@ -167,6 +169,25 @@
+ #if HAVE(PTHREAD_NP_H) || OS(NETBSD)
+     // e.g. on FreeBSD 5.4, neundorf@kde.org
+     pthread_attr_get_np(thread, &sattr);
++#elif defined(__linux__) && !defined(__GLIBC__)
++        {
++            unsigned long inStack;
++            FILE *maps;
++            
++            inStack = (unsigned long) &inStack;
++            maps = fopen("/proc/self/maps", "r");
++            if (maps != NULL) {
++                unsigned long start, end;
++                while (fscanf(maps, "%lx-%lx%*[^\n]\n", &start, &end) >= 2) {
++                    if (start <= inStack && end > inStack) {
++                        // found it
++                        stackBase = (void *) end; // FIXME: assuming we grow down
++                        break;
++                    }
++                }
++                fclose(maps);
++            }
++        }
+ #else
+     // FIXME: this function is non-portable; other POSIX systems may have different np alternatives
+     pthread_getattr_np(thread, &sattr);
