# HG changeset patch
# Parent e585f3533fca60d1a8e5473b8a99615b6a051640

diff -r e585f3533fca x11/libXfont/Makefile
--- a/x11/libXfont/Makefile	Tue Oct 09 17:56:57 2012 -0400
+++ b/x11/libXfont/Makefile	Tue Oct 09 17:57:02 2012 -0400
@@ -20,6 +20,7 @@
 PKGCONFIG_OVERRIDE+=	xfont.pc.in
 
 CONFIGURE_ARGS+=	--disable-devel-docs
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -DNOFILES_MAX=256"
 
 .include "../../meta-pkgs/modular-xorg/hacks.mk"
 
