# HG changeset patch
# Parent c504a0f83c59bfb0f7a3fc043d638953d99933cb

diff -r c504a0f83c59 x11/motif/Makefile
--- a/x11/motif/Makefile	Mon Jul 08 19:26:47 2013 -0400
+++ b/x11/motif/Makefile	Mon Jul 08 19:26:53 2013 -0400
@@ -21,6 +21,9 @@
 
 CONFLICTS+=		lesstif-[0-9]* openmotif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config automake yacc
 GNU_CONFIGURE=		yes
