# HG changeset patch
# Parent 3d22f8b201de168dd208fff09f349b1998590f50

diff -r 3d22f8b201de x11/libXfont/Makefile
--- a/x11/libXfont/Makefile	Mon Feb 18 12:27:52 2013 -0500
+++ b/x11/libXfont/Makefile	Mon Feb 18 12:27:57 2013 -0500
@@ -18,6 +18,7 @@
 PKGCONFIG_OVERRIDE+=	xfont.pc.in
 
 CONFIGURE_ARGS+=	--disable-devel-docs
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -DNOFILES_MAX=256"
 
 .include "../../meta-pkgs/modular-xorg/hacks.mk"
 
