# HG changeset patch
# Parent b5ded3062f1bc3f898ab9e6782534b7564c36f12

diff -r b5ded3062f1b x11/pixman/Makefile
--- a/x11/pixman/Makefile	Tue Oct 09 17:57:21 2012 -0400
+++ b/x11/pixman/Makefile	Tue Oct 09 17:57:26 2012 -0400
@@ -18,5 +18,8 @@
 
 PKGCONFIG_OVERRIDE+=	pixman-1.pc.in
 PKGCONFIG_OVERRIDE+=	pixman-1-uninstalled.pc.in
+	
+pre-configure:
+	sed -i 's/support_for__thread=yes/support_for__thread=no/' ${WRKSRC}/configure	
 
 .include "../../mk/bsd.pkg.mk"
