# HG changeset patch
# Parent 7d46cec7f7ad633e51365a8d2c7c7954b333d14d
diff -r 7d46cec7f7ad -r 0e0bb4bda941 x11/openmotif/Makefile
--- a/x11/openmotif/Makefile	Thu Nov 22 02:01:46 2012 -0500
+++ b/x11/openmotif/Makefile	Thu Nov 22 02:16:59 2012 -0500
@@ -38,6 +38,9 @@
 
 CONFLICTS+=		lesstif-[0-9]*
 
+# [Gregor] Another weird libiconv issue
+CFLAGS+=		-liconv
+
 USE_LIBTOOL=		yes
 USE_TOOLS+=		flex gmake tbl pkg-config
 GNU_CONFIGURE=		yes
