# HG changeset patch
# Parent ee731725f02320485238cd53a46937d1531d05e9

diff -r ee731725f023 x11/libFS/Makefile
--- a/x11/libFS/Makefile	Mon Jul 08 19:23:12 2013 -0400
+++ b/x11/libFS/Makefile	Mon Jul 08 19:23:18 2013 -0400
@@ -25,6 +25,8 @@
 
 .include "../../meta-pkgs/modular-xorg/hacks.mk"
 
+CONFIGURE_ARGS+=	CFLAGS="-O2 -g -DNOFILES_MAX=256"
+
 .include "../../x11/fontsproto/buildlink3.mk"
 .include "../../x11/xproto/buildlink3.mk"
 .include "../../x11/xtrans/buildlink3.mk"
diff -r ee731725f023 x11/libFS/distinfo
--- a/x11/libFS/distinfo	Mon Jul 08 19:23:12 2013 -0400
+++ b/x11/libFS/distinfo	Mon Jul 08 19:23:18 2013 -0400
@@ -3,3 +3,4 @@
 SHA1 (libFS-1.0.5.tar.bz2) = 3a94bc42775f4aa2eac14a51e0043299d7cd31b6
 RMD160 (libFS-1.0.5.tar.bz2) = c3f6bb8fed4ac1904e55fe36cb4650e69f37dc06
 Size (libFS-1.0.5.tar.bz2) = 303806 bytes
+SHA1 (patch-aq) = 75f64d23fe0ffdc5dca48fcba982539d34caf42b
diff -r ee731725f023 x11/libFS/patches/patch-aq
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/x11/libFS/patches/patch-aq	Mon Jul 08 19:23:18 2013 -0400
@@ -0,0 +1,12 @@
+diff -r 5f55f742a811 src/FSConnServ.c
+--- src/FSConnServ.c	Mon May 07 19:40:35 2012 +0000
++++ src/FSConnServ.c	Mon May 07 19:44:53 2012 +0000
+@@ -49,6 +49,8 @@
+ 
+ */
+ 
++#define _GNU_SOURCE
++
+ #ifdef HAVE_CONFIG_H
+ #include <config.h>
+ #endif
